\documentclass[12pt]{"article"}
\setlength{\parindent}{0pt}
\linespread{1.5}
\usepackage[margin=1.5in]{geometry}
\usepackage{graphicx}
\graphicspath{ {../Figures/} }
\usepackage{titlesec}
\newcommand{\sectionbreak}{\clearpage}
\usepackage{sidecap}
\usepackage[singlelinecheck=false, labelfont=bf]{caption}
\usepackage{verbatim}
\usepackage{setspace}
\newcommand{\mycaption}[2]{\caption[#1]{\textbf{#1.} #2}}
\usepackage{chngcntr}
\counterwithin{figure}{section}
\usepackage{hyperref}
\usepackage{amsmath}
\usepackage{comment}
\usepackage{mhchem}
\usepackage{cleveref}
\usepackage{caption}
\captionsetup{font=footnotesize}
\usepackage[backend=bibtex,style=authoryear, natbib=true]{biblatex} %
\addbibresource{bibliography.bib}


\begin{document}

\begin{titlepage}
\centering
{\huge Title\\}
\end{titlepage}


\pagebreak

\begin{Large}
\textbf{Abstract}\\
\end{Large}


\pagebreak

\begin{Large}
\textbf{Impact statement}\\
\end{Large}


\pagebreak

\begin{Large}
\textbf{Acknowledgements}\\
\end{Large}


\tableofcontents
\listoffigures
\listoftables


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\section{Introduction}

\subsection{Spatial patterning in biological systems}

\clearpage
\subsection{Cell polarity}
\subsubsection{The Min system}
\subsubsection{CDC-42 polarity in yeast}
\subsubsection{PAR polarity}

A final example of cell polarity is PAR polarity. Best studied in the developing C elegans embryo, PAR polarity plays an essential role in development by regulating a series of asymmetric cell division. PAR proteins form patterns on the cell membrane which direct asymmetric localisation off the spindle and asymmetric distribution of fate determinants. The resulting cell division is asymmetric, leading to two cells that differ in size and fate.\\

The PAR network is a highly conserved network of proteins that drives polarity in a range of cell types across the animal kingdom (\cite{Goldstein2007}). PAR polarity plays a number of essential roles throughout development, including asymmetric cell division, cell fate specification, regulation of tissue architecture and cell motility. Key to this process is the formation of asymmetric PAR protein patterns at the cortex of cells. PAR polarity is distinct from the two other examples described so far as it relies on two distinct groups of proteins, referred to as anterior and posterior PARs (aPARs and pPARs) that show distinct and complementary cortical patterns. Rather than self-amplification, in which one protein (or group of proteins) directly feeds back onto its own local enrichment, the dominant patterning mechanism in PAR polarity is one of mutual antagonism, whereby proteins within the two subgroups negatively regulate each-other’s localisation \parencite{Goehring2011b}. \\

A classic model system for studies of the PAR network is the C. elegans zygote. PAR proteins were first found to be important for this process in a screen of mutants that disrupt asymmetric zygote divisions (Kemphues and others… Tabuse, Watts). Work in the following decades has gone to characterize the PAR proteins as a diverse and highly conserved set of scaffold proteins, adaptors and enzymes, some which play essential roles in the self-organisation of PAR polarity \parencite{Lang2017}. Key to this process is cross-phosphorylation of the two subgroups, driven by kinases within each group, which negatively regulates membrane association and keeps the domains separate (refs).\\

Once PAR polarity is set up, enzymes within each group direct a set of downstream processes, leading to asymmetric segregation of cell fate determinants \citep{Cuenca2003, Daniels2010, Griffin2011, Wu2018} and asymmetric placement of the cleavage plane \parencite{Bouvrais2018}. The net result is an asymmetric cell division, with daughter cells differing in both size and fate. This process repeats in subsequent cell divisions, playing an essential role in early patterning of the embryo (fig x). \\

INSERT FIGURE \\

As well as the C elegans embryo, PAR protein homologies display asymmetric localization in many different organisms and cell types, where they play a number of essential roles. <MORE NEEDED HERE, e.g. neuroblasts. Mechanisms may differ, such as unipolar polarity without an opposing domain in some cases>.\\


\clearpage
\subsection{Maintenance of cell polarity by bistable reaction-diffusion systems}
\subsubsection{Bistable reaction kinetics}
\subsubsection{Single species polarity models}
\subsubsection{The mutual antagonism model}

\clearpage
\subsection{A molecular basis for ultrasensitivity}
\subsubsection{Ultrasensitivity in protein phosphorylation reactions}
\subsubsection{Ultrasensitivity through cooperative membrane binding}

\clearpage
\subsection{PAR polarity in C elegans zygotes}

Intro to section

\begin{figure}[!h]
\includegraphics[scale=0.9]{par_proteins_schematic}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:par_proteins_schematic}
\end{figure}


\subsubsection{Mechanisms of PAR cortical association}

TRANSITION TO SECTION. PAR proteins associate to the cortex in a number of ways. Some display direct and intrinsic cortical localisation activity (PAR-3, PAR-2, CDC-42, CHIN-1, LGL-1). Whilst originally thought to interact with the cytoskeleton, these PARs are now known to localise to the cortex by interacting with the plasma membrane through a combination of electrostatics and specific phospholipid interactions. Other PARs lack intrinsic cortical localisation activity (PAR-6, PKC-3), relying instead on other PARs to act as scaffolds and adaptors, or fall somewhere in between with a mix of direct and scaffold-mediated binding (PAR-1). In this section I will overview some of the key mechanisms.\\


\textit{Association with lipids}\\

% make distinction between lipid interactions and charge based early on

PAR-3, associates with the membrane via its PDZ2 domain \parencite{Li2010a}. <MORE, any research on PDZ2 domains, what is this binding to?>.  The C-terminal portion also plays a role in promoting strong membrane association, although the function of this region isn’t understood (?, I think more is known about it in other species) (Li 2010b). \\

The membrane localisation of CDC-42 is largely due to a c-terminal geranylgeranyl moiety (REF). MORE HERE. The protein additionally contains a conserved cluster of positively charged residues directly preceding the geranylgeranyl moiety, including a di-arginine motif which promotes specificity for PIP2 containing membranes \parencite{Johnson2012}.\\

Cortical localisation of PAR-2 in vivo depends on a central unstructured region of the protein rich in basic amino acids \parencite{Hao2006}. Full-length PAR-2 displays an ability to bind to an array of positively charged phospholipids in vitro, suggesting an electrostatics-based interaction rather than specific interaction with any one phospholipid \parencite{Motegi2011}. Given this promiscuous nature, its apparent specificity for the plasma membrane in vivo is poorly understood, but may reflect the increased charge associated with this membrane compared to other membranes (REF).\\

PAR-1 contains a C-terminal KA domain, a common membrane association domain, which can bind to membranes and (similarly to PAR-2) interact non-specifically with anionic phospholipids \parencite{Moravcevic2010}. This domain has been shown to be both necessary and sufficient for cortical localisation in vivo \parencite{Motegi2011}.\\

Cortical association of LGL-1 relies on a region towards the C-terminus of the protein, which is rich in positively charged amino acids and can directly bind to negatively charged membranes \parencite{Visco2016}. Independent of overall membrane charge, affinity is strongest for membranes enriched in diphosphoinositides (Visco, NEED TO CHECK), which are most abundant in the inner leaflet of the plasma membrane. Upon membrane binding, the membrane binding domain folds into an alpha-helix, creating a positively charged patch of basic amino acids. Mutations at some, but not all of these basic amino acids, lowers affinity for diphosphoinositides, suggesting that this folded domain is important for membrane binding specificity. \\


\textit{Interaction with scaffolds}\\

Other PARs lack intrinsic cortical localisation activity, relying instead on other PARs to act as scaffolds and adaptors. PAR-6 and PKC-3 are stable binding partners, interacting via PB1 domains at the N-terminus of each protein (\cite{Hirano2005}), and in normal circumstances are dependent on each other for stable cortical association (refs: Hung, Tabuse). Proper cortical association of this complex relies on interactions with both PAR-3 and CDC-42. PAR-6 interacts with CDC-42 via its semi-CRIB domain, which is a requirement for proper cortical association (\cite{Aceto2006}). It can also interact with PAR-3 via the PDZ1 domain of PAR-3 and the PDZ domain of PAR-6 (Li 2010a). However, this interaction doesn’t appear to play an essential role in vivo, as mutations to this domain which disrupt the interaction in vitro have no effect on PAR-6 localisation in vivo. PKC-3 engages with PAR-3 via its kinase domain. Two sites flanking the phosphosite direct binding to the CR3 domain of PAR-3 \citep{Soriano2016}). Upstream of the phosphosite is an FxR site, a conserved motif found in PKC-3 substrates which provides an anchor point for PKC-3. Downstream is a hook motif which engages pockets within the PKC-3 kinase domain and disrupt an N-lobe required for catalytic activity, keeping PKC-3 in an inactive state.\\

In zygotes, PAR-6/PKC-3 accumulate at the membrane in two distinct pools: a punctate PAR-3 dependent pool, and a diffuse CDC-42 dependent pool \citep{Aceto2006}\citep{Beers2006}. The punctate pool represents PAR-6/PKC-3 directly associated with PAR-3 \citep{Dickinson2017}, whereas the diffuse pool represents PAR-6/PKC-3 bound to CDC-42. PAR-3 is usually essential for any PAR-6/PKC-3 localisation, suggesting that the PAR-3 associated state is a prerequisite for assembly into the CDC-42 associated state. Interestingly, however, inhibition of PKC-3 kinase activity allows the complex to bypass this requirement and interact with CDC42 directly in the absence of PAR-3 \citep{Rodriguez2017}. This implies a model where the complex is first recruited into a PAR-3 associated complex <via what interactions?>, PKC-3 is then inactivated by PAR-3 <need details on this: see Soriano>, which permits transfer to a CDC-42 associated state, in which the inhibition of PKC-3 is relieved.\\

Whilst PAR-1 displays some intrinsic lipid binding activity, it’s cortical localisation is largely mediated by interaction with PAR-2. Again, this interaction is via the KA domain of PAR-2, which interacts with an unknown region of PAR-2 \citep{Motegi2011}. This reaction leads to local recruitment of PAR-1 by PAR-2, and cortical localisation in regions of PAR-2 enrichment.\\


\textit{Self-association and clustering}\\

For some PAR proteins, a key determinant for stable association is the ability to self-associate into oligomers, in some cases forming large clusters. PAR-3 contains a CR1 domain at the N-terminus, an oligomerisation domain which assembles into helical filaments in vitro (Feng, Zhang). Oligomerisation of the protein via this domain is essential for stable membrane association in vivo \citep{Dickinson2017} (others: Li 2010, Benton, Feng, Mizuno), although CR1 domain mutants can display transient cortical association (REF). <Comment on this>. Clustering is negatively regulated by PLK-1 phosphorylation, which conveys cell cycle dependence on PAR-3 cortical association. <MORE, REFS, Dickinson>\\

Whilst little is known about the mechanisms of CHIN-1 cortical association, it has also been observed to localise in discrete puncta \citep{Kumfer2010}. These punca only appears during late maintenance phase. It’s therefore plausible that, similar to PAR-3, self-association might be under regulatory control.\\

PAR-2 has also been suggested to self-associate and form clusters on the cortex \citep{Arata2016}. <leave main discussion until later>\\


\subsubsection{Maintenance of polarity by mutual antagonism}

Polarity occurs when the cortical localisation of the PARs is biased, such that the aPARs accumulate on the membrane on one side of the cell, and pPARs on the other. A key component of this is mutual antagonism, whereby proteins within each of the two groups are able to negatively regulate the cortical association of proteins in the other group. There are a number of redundant pathways.\\

\textit{PKC-3 pathway}\\ 

Antagonism from aPARs to pPARs is driven exclusively by PKC-3. <FxR, see Soriano>. Phosphorylation adds negative charge to the proteins, which repels them from the membrane. MORE HERE\\

In the case of PAR-1 and LGL-1, phosphorylation is at a single site within the membrane association domain \citep{Hoege2010}\citep{Motegi2011}. Phosphorylation of these sites is necessary and sufficient to exclude these proteins from the membrane. PAR-2 is more complex, with seven predicted phosphorylation sites \citep{Hao2006}, although these have yet to be verified biochemically. Some of these are within the membrane association domain, and likely regulate membrane association in an analogous way to PAR-1 and LGL-1. Phosphorylation by PKC-3 has been shown to disrupt binding of PAR-2 to phospholipids in vitro, which is thought to represent reduced electrostatic attraction via the addition of negatively charged phosphate groups \citep{Motegi2011}. Some of the predicted phosphorylation sites are outside the membrane association region and may play regulatory or gatekeeping roles. Mutation of all seven sites prevents phosphorylation in vivo, leading to uniform PAR-2 \citep{Hao2006}). Whilst individual mutant analysis hasn’t been performed on all of these sites independently, \textcite{Motegi2011} show that mutation of just one of these sites (S241A), which falls outside of the main membrane localisation region (CHECK), significantly reduces phosphorylation in vitro and achieves the same phenotype as the 7S/E mutant in vivo, indicating that this site may play a key role as a gatekeeper. Phosphomimetic mutation of all seven sites weakens, but doesn’t completely eliminate membrane association, indicating that there may additional sites on the protein, or that these mutations fail to completely mimic phosphorylation (what’s the charge of E compared to a phosphate?). \\

CHIN-1 doesn’t have an FxR site (check), but is also excluded from the anterior by PKC-3 \citep{Sailer2015}. This is thought to involve direct inhibition of CHIN-1 clustering at the cortex by PKC-3, however the mechanistic basis of this is poorly understood.\\

\textit{PAR-1 pathway} \\

PAR-1 can phosphorylate PAR-3, which it does primarily at a single serine (S950) towards the C-terminus of the protein \citep{Motegi2011}. <REGULATES CLUSTERING> In a wild-type background, depletion of PAR-1, or mutation of the phosphosite on PAR-3, causes PAR-3 to associate with the posterior cortex, although this association is weak \citep{Sailer2015}. It is unclear why some degree of asymmetry is maintained in these conditions, although this may be a remnant of earlier transport by cortical flows and relatively stable cortical association which prevents lateral diffusion and cortical-cytoplasmic exchange that would redistribute the protein. If cortical flows are inhibited, PAR-1 mutation prevents aPARs from polarising at all \citep{Motegi2011}.\\

\textit{LGL-1 pathway}\\

LGL-1 has been proposed to antagonise aPARs by forming a complex with PAR-6/PKC-3, the whole of which dissociates from the cortex after LGL-1 is phosphorylated by PKC-3 \citep{Hoege2010}. LGL-1 loss has no observable effects in zygotes in otherwise wild type systems, indicating that this is usually of minor importance, but can enhance phenotypes in PAR-2 mutants \citep{Beatty2010}. Furthermore, LGL overexpression is able to compensate for absence of PAR-2 \citep{Hoege2010}, indicating that this pathway can be sufficient to take over the roles of the PAR-2/PAR-1 pathway.\\

\textit{CHIN-1 pathway}\\

CHIN-1, a GAP for CDC-42 appears on the posterior cortex late in the cell cycle and restricts CDC-42 activity to the anterior \citep{Kumfer2010} \citep{Beatty2013}\citep{Sailer2015}. CHIN-1 loss results in uniform CDC-42 activity, but this doesn’t lead to uniform PAR-6/PKC-3 localisation (Sailer, others), indicating that active CDC-42 isn’t sufficient to recruit PAR-6/PKC-3. When combined with a PAR-1 mutant, however, which leads to a small amount of PAR-3 binding in the posterior, PAR-6/PKC-3 is not recruited to a high level in the posterior. This implies that PAR-3 gates association with CDC-42. PAR-3 asymmetry is required to restrict this gating to the anterior, and CDC-42 asymmetry is required to restrict the binding partner of PAR-6/PKC-3 to the anterior, although either one of these behaviours is sufficient.\\


\subsubsection{Establishment of polarity}

As previously mentioned, PAR polarity doesn’t occur spontaneously, requiring a trigger so that polarity happens at the correct time and with the correct orientation. Prior to the establishment of asymmetry, aPARs are uniformly enriched on the cortex, and exclude pPARs through antagonistic interactions. Polarity is then triggered by signals from the MTOC which forms near the site of sperm entry, via two redundant pathways:\\

\textit{Anterior-directed cortical flows}\\

A first mechanism involves a local inhibition of RhoA activity in the posterior of the cell, which downregulates actomyosin contractility. This sets up a spatial gradient of contractility, which leads to anterior-directed cortical flows. The aPARs PAR-3, PAR-6 and PKC-3 are able to tap into these flows, which causes them to segregate to the anterior. <more here on local RhoA inhibition, par-3 clustering>\\

The resulting depletion of PKC-3 concentration in the posterior relieves antagonism of pPARs, allowing them to bind in the posterior and form a nascent domain. This domain is then amplified via a series of antagonistic feedback reactions, which are described in another section. This mechanism of advective triggering followed by self-organisation is sufficient to capture the core features and dynamics of polarity in computer models of the PAR network (Goehring).\\

Further work has shown that the PAR proteins themselves are able to feed back onto the actomyosin cortex, amplifying contractility asymmetries as polarity progresses \citep{Gross2018} <More here>.\\

Whilst primarily important during establishment phase, continued regulation of cortical flow by the PAR proteins also plays a role in preventing the breakdown of polarity during maintenance phase. In par-2 mutants, whilst anterior-directed cortical flow proceeds as normal during polarity establishment (Gross CHECK), these are followed by aberrant posterior-directed cortical flows at maintenance phase, leading to significant spread of aPARs (ref). The precise mechanistic reasons for this misregulation in PAR-2 mutants is unclear, but it suggests that signalling from PAR-2 plays a role some preventing rearwards cortical flows at maintenance phase.  PAR-2 loss leads to higher cortical myosin accumulation independently of PAR-6/PKC-3 presence \citep{Munro2004}\citep{Beatty2013}, suggesting that this isn’t an indirect effect via PKC-3 signalling. Backwards flows can be prevented by mutation of MRCK-1, <details on this protein>.\\

LGL-1 can also regulate maintenance phase flows in a similar way. Dual loss of PAR-2 and LGL-1 leads to stronger rearwards flow \citep{Beatty2010}, and overexpression of LGL-1 is able to rescue rearwards flow in PAR-2 mutants \citep{Hoege2010}. However, unlike PAR-2, LGL-1 has no effect on myosin accumulation in the absence of PAR-6, suggesting that this LGL-1 dependent effect may be indirect via aPARs and the known roles of PKC-3 in cytoskeleton control (i.e. LGL-1 loss weakens antagonism, leading to aPAR invasion, which redistributes flows).\\

\textit{The microtubule pathway}\\

In the absence of cortical flows, symmetry breaking still occurs, albeit later, indicating the existence of multiple symmetry breaking mechanisms. A second triggering mechanism involves an interaction between PAR-2 and microtubules emanating from the sperm-donated centrosome. In no flow regimes, PAR-2 symmetry breaking occurs late, and correlates spatially and temporally with the site of MTOC-cortex contact \citep{Motegi2011}. Treatments that disrupt microtubules prevent symmetry breaking in these conditions.\\

Mechanistically, this is carried out by a direct interaction between PAR-2 and microtubules, which is thought to shield the phosphorylation sites on PAR-2, and has been shown to reduce phosphorylation by PKC-3 in vitro. This creates a zone of local protection in the posterior of the cell, allowing PAR-2, and thus PAR-1 to load, kicking of self-organisation. Mutantions at the microtubule binding interface on PAR-2 have a similar phenotype to nocodazole treatments.\\

\textit{Additional pathways}\\

Additional mechanisms may underlie triggering in some circumstances. <Evidence of still breaking symmetry when other cues are lost>. Microfabrication studies show that PAR-2 has a preference for curved membranes. This may lead to preferential binding of PAR-2 at the poles of cells, which has been suggested to act as a symmetry breaking cue in cases where normal symmetry breaking is misregulated \citep{Klinkert2019}. The mechanistic basis for this proposed curvature sensitivity is unclear.\\


\subsubsection{Downstream of the PAR proteins}

The primary role of the PAR proteins is to regulate an asymmetric cell division, whereby the P0 divides to give two cells differing in size and fate. This is carried out via two pathways:\\

\textit{Placement of the division plane}\\

Signalling from the PARs regulates the position of the mitotic spindle, which leads to a cell size asymmetry following cytokinesis. This is carried out predominantly by PKC-3, which phosphorylates <> LIN-5. This results in decreased microtubule pulling forces in the anterior, where PKC-3 is highest, leading to a shift in the position of the mitotic spindle towards the posterior. PAR-2 also has a direct effect on spindle pulling forces through an unknown mechanism, independently of aPARs, which may contribute to placement of the division plane (Rodrigues).\\


\textit{Segregation of fate determinants}\\

As well as differing in size, the two daughter cells differ in a number of cytoplasmic components which define cell fate during development, which is also set up by signalling from the PARs. Immediately downstream of the PARs is MEX-5, which is organised into a cytoplasmic gradient of MEX-5 in response to asymmetry of PAR-1. PAR-1 phosphorylates MEX-5 \citep{Griffin2011}, which increases its mobility. Working against the action of a uniform phosphatase, PP2A, this leads to an asymmetry in MEX mobility, which leads to accumulation at the anterior where mobility is lowest. <Computer models, Griffin>. \\

This MEX gradient then sets up a P granule asymmetry by regulating growth and dissolution of phase-separated P-granule droplets \citep{Brangwynne2009}. These granules dominate in the posterior, so are inherited by the P1 cell after cell division. The granules contain fate determinants which are responsible for specifying germ-line fate in the P lineage.\\


\subsubsection{Resistance and substrate competition}

\subsubsection{Discussion}

\clearpage
\subsection{PAR-2: roles and mechanisms of action}
\subsubsection{Main functional roles of PAR-2}

To summarise some of the information in the previous sections, the main roles for PAR-2 are as follows:\\

\textbf{Recruitment of PAR-1}. This plays a key role in preventing PAR-3 from associating to the cortex in the posterior, although other mechanisms contribute to keeping PAR-3 in the anterior, which isn’t understood (refs). This acts redundantly with the CHIN-1 pathway to keep the kinase PKC-3 out of the posterior (Sailer, others), and LGL-1 may also contribute to this (Hoege, Beatty, others). \\

Interestingly, despite the role for PAR-1 in segregation of fate determinants, PAR-2 this appears dispensable for proper segregation of fate determinants in in the zygote. Whilst absent from the cortex in these conditions, PAR-1 is still able to maintain a cytoplasmic concentration and activity gradient (mechanism?), meaning that MEX-5 and P-granule asymmetry are largely intact. Notably, however, localisation of fate determinants is impaired at later stages in the embryo in these conditions. Thus, a primary function of the PAR-1/PAR-2 interaction may be to ensure that PAR-1 is segregated and enriched through the germ line, so that downstream signalling can continue in, and be restricted to, the developing P-lineage. \\

\textbf{Protection of PKC-3 substrates}. PAR-2 has been demonstrated to protect both PAR-1 and itself from the antagonistic action of PKC-3 (Motegi, others). In the case of PAR-1, this involves direct shielding of the phosphorylation site on PAR-1 as well as competitive inhibition \citep{Ramanujam2018}. It’s plausible that a similar competitive inhibition mechanism may operate to protect other PKC-3 substrates, namely LGL-1 and CHIN-1, although this hasn’t been formally tested. In the case of cortical PAR-2 being able to protect other PAR-2 molecules, the mechanism isn’t clear. Competitive inhibition may contribute, and the RING domain of the protein has also been implicated in self-recognition, as described below. \\

\textbf{Restriction of cortical flows at maintenance phase}. PAR-2 appears to play a direct role in preventing rearwards flows at maintenance phase which would otherwise redistribute aPARs to the posterior. The mechanistic basis of this is not understood.\\

\textbf{Signalling to the mitotic spindle}. May contribute to placement of the division boundary, although the mechanism is unclear.\\


\subsubsection{Evidence and proposed roles for oligomerisation}

As previously mentioned, PAR-2 has been suggested to oligomerise, which may stabilise membrane association. This claim is based primarily on a study by \textcite{Arata2016}. By TIRF imaging, the authors were able to resolve distinct PAR-2 particles of varying size at the cortex, which showed a slight asymmetry towards larger oligomers at the posterior. Based on fluorescence intensity, the largest particles were estimated to be at least tetrameric. Membrane lifetime was found to vary across the cell according to oligomer size and local PKC-3 concentration, indicating that oligomerisation can increase, and phosphorylation decrease, stability of membrane binding. They also found an asymmetry in the membrane association rate, highest in the posterior of polarised cells, which they suggest could be due to direct recruitment of cytoplasmic PAR-2 into cortical oligomers.\\

This ability to self-associate is supported by in vitro pull-down studies, which show that tagged PAR-2 is able to pull down untagged PAR-2 \citep{Motegi2011}\citep{Arata2016}. These reports show that the amount pulled down is very small, indicating a weak, non-constitutive interaction. However, quantitative measurements of dimer affinity and oligomer size using biophysical methods have not been performed.\\

Taken together, \textcite{Arata2016} propose that an initial PKC-3 asymmetry leads to a PAR-2 oligomer size asymmetry, either as a direct effect of phosphorylation disrupting oligomerisation, or as a result of concentration-dependent oligomer growth/dissociation. Oligomer size asymmetry would in turn stabilise concentration asymmetries, through on and off rate effects, leading to a degree of positive feedback and polarity stabilisation.\\

Whilst the core-concepts of their model (namely concentration/phosphorylation dependent oligomerisation, oligomer-size dependent membrane stability, and self-recruitment) are attractive and intuitive, they have yet to be formalised by mechanistic computer models in the context of PAR-2 polarity. Furthermore, the mechanistic basis of the putative PAR-2 oligomerisation reaction remains elusive.\\


\subsubsection{Roles for the RING domain}

Studies of mutant PAR-2 alleles have demonstrated an important role for the N-terminal RING domain in establishing strong PAR-2 domains. Mutations to the putative zinc coordinating residues in the RING domain, designed to misfold the domain and render it non-functional, weaken the strength of PAR-2 cortical localisation \citep{Hao2006}. This is accompanied by faster cortical dynamics as revealed by FRAP, indicative of a shorter membrane lifetime \citep{Motegi2011}. Whilst these mutants can form posterior domains at establishment phase (albeit weakly concentrated), they are rapidly cleared by invading aPARs at maintenance phase \citep{Hao2006}.\\

The mechanistic basis of this mutant phenotype is poorly understood, and perhaps surprising given that the RING domain is distinct from the cortical localisation domain of the protein and shows no cortical binding activity in isolation \citep{Hao2006}. RING mutants show reduced membrane affinity even in the absence of aPARs, indicating that the phenotype is, at least partially, intrinsic to PAR-2, rather than through increased sensitivity to aPARs. It is also unlikely that reduced membrane association is entirely a secondary consequence of the unfolded domain in cysteine mutants, as truncation mutants lacking the domain show a qualitatively similar phenotype, although a quantitative comparison hasn’t been performed.\\

Studies on RING mutants also shed potential light on the mechanisms of PAR-2 protection. \textcite{Hao2006} showed that RING mutants are unable to be protected by endogenous PAR-2 in PAR-1 mutant conditions in which aPARs become uniform, suggesting that the RING may be required to react to endogenous PAR-2. Whilst unclear at the time of the study, more recent work showing that PAR-2 can oligomerise, as described above, along with a characteristic role of RING domains in oligomerisation, as described below, may shed potential light on this. However, this hypothesis is yet to be formally tested. Intriguingly, however, Motegi showed that in near identical experimental conditions RING mutant PAR-2 can (seemingly) be protected by endogenous PAR-2. The reason behind this discrepancy is unclear.\\


\subsubsection{Discussion}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\section{A pipeline for quantification of membrane and cytoplasmic protein concentrations}

Intro to section, build on narrative from previous section

\subsection{Autofluorescence correction}

\textit{Note: this section has been adapted from \textcite{Rodrigues2022}, and describes work performed with Nelio Rodrigues.}

\subsubsection{Autofluorescence in \textit{C. elegans}}

One major barrier in quantitative experiments using \textit{C. elegans} is autofluorescence (AF), which is particularly prominent in channels excited with blue wavelengths which are commonly used to image green fluorophores. When using endogenously tagged proteins, which are often expressed at low levels, this contribution can often be a significant fraction of the total signal, and can therefore significantly obscure the true signal that one is interested in. This might pose particular problems for quantitative experiments, where the absolute signal levels may be important.\\

We can observe the significance of the problem in \textit{C. elegans} by imaging untagged control embryos. As shown in \cref{fig:saibr_n2_vs_lgl} (left panels), a significant amount of signal is collected in the regular GFP channel (488nm excitation, 535/50nm emission), which varies both spatially within the image, and between different images. By comparison, total signal in embryos endogenously tagged with LGL-1::GFP ( \cref{fig:saibr_n2_vs_lgl}, right panels) is also highly variable, and only marginally higher than N2s, suggesting that a significant fraction of the total signal observed in these cells is autofluorescence, and that much of the intra-embryo signal variation is likely due to variable autofluorescence. Despite being enriched at the posterior membrane, which is easily visible in cells with overexpressed LGL-1 (e.g. \cite{Hoege2010}), this is difficult to visualise here as a result of autofluorescence. Therefore, if we want to accurately visualise, and indeed quantify, protein levels and distributions, we need a method that can locally correct AF on a pixel-by-pixel basis.\\

\begin{SCfigure}
\includegraphics[scale=0.9]{saibr_n2_vs_lgl}
\mycaption{Autofluorescence is abundant and variable in images of \textit{C. elegans} zygotes}{GFP channel (488nm excitation, 535/50nm emission) images of unlabelled N2 and LGL-1::GFP embryos taken under identical imaging conditions. Pixel scaling is the same for each image. Fig. 1B from \textcite{Rodrigues2022}.}
\label{fig:saibr_n2_vs_lgl}
\end{SCfigure}

One approach that has been used to tackle autofluorescence in some systems is spectral imaging \citep{Billinton2001}. Typically used to separate overlapping fluorophore signals based on spectral characteristics, this approach can also be used to separate out autofluorescence by treating it much like a fluorophore with its own spectral characteristics. Whilst often effective, these techniques require specialised instruments and analysis tools and cannot be performed on standard confocal microscopes.\\

A simpler, related, approach has been proposed for some systems (e.g. \cite{Roederer1986}) which exploits the fact that autofluorescence can often be described as a single fluorescent component, with an emission spectrum much broader than GFP. If one can find an emission wavelength (usually red shifted) that is specific for autofluorescence, this can be used to infer the amount of autofluorescence in the sample, which can then be subtracted away from the regular fluorophore channel to give a clean readout of fluorophore signal. In comparison to full spectral imaging, this method can be carried out with standard light sources and emission filters, and therefore can be easily implemented into existing workflows.\\

Inspired by this approach, we aimed to implement, and assess the applicability of such a method for autofluorescence removal in images of \textit{C. elegans} embryos. In doing so, we have put together a robust and easily-implementable workflow which we’ve termed SAIBR: Spectral Autofluorescence Image correction by Regression.\\


\subsubsection{SAIBR: a simplified method for autofluorescence correction based on dual emission imaging}


At minimum, autofluorescence correction relies on the ability to find a reporter channel that is free of GFP signal, but rich in autofluorescence, such that this channel can be used as an independent readout of autofluorescence in the sample. Full spectral analysis performed by Nelio Rodrigues (not shown here), shows that a channel with a red shifted emission filter, such as those commonly used to image red fluorescent proteins, meets such a requirement.\\

Furthermore, by imaging untagged embryos with both the standard GFP channel and this red-shifted autofluorescence-reporter channel (488nm excitation, 630/75nm emission), which I will refer to as the AF channel, we find a strong linear correlation between pixel data from the two channels (\cref{fig:saibr_n2_correlation}). Whilst raw pixel values do not correlate well, as these are dominated by noise, we can get a strong correlation by first applying a Gaussian filter to suppress this noise (\cref{fig:saibr_n2_correlation}A). We found that this relationship is consistent between embryos (\cref{fig:saibr_n2_correlation}B, C). Furthermore, we found a near identical relationship when plotting the mean intensity values of individual embryos, suggesting that the same relationship can account for both intra- and inter-embryo AF variation. \\

\begin{figure}
\includegraphics[scale=0.95]{saibr_n2_correlation}
\centering
\mycaption{Autofluorescence is spatially correlated across emission channels}{
\textbf{(A)} Correlation of pixel values between the GFP channel and AF channel for an unlabelled N2 embryo, subject to Gaussian blur of indicated radius. Pixel values were taken from an ROI encompassing the entire embryo and a portion of the background. All pixels within this region were used for the regression, but for clarity only a random 10\% sample of pixels are shown.
\textbf{(B)} Comparison of inter-channel pixel correlation (Gaussian radius = 1) for three unlabelled N2 embryos, colour coded by embryo. Histograms of intensity values for GFP and AF channels shown for reference.
\textbf{(C)} Comparison of per-pixel correlation with data obtained from whole embryo means. (i) Lines indicate per-pixel regression for individual embryos as in (B). (ii) Overlay showing mean whole embryo fluorescence values (circles).
Fig. S2 and 1E-F from \textcite{Rodrigues2022}.
}
\label{fig:saibr_n2_correlation}
\end{figure}

Together, this implies that taking an AF channel image should be sufficient to accurately predict the level of autofluorescence in the GFP channel, and thus subtract it out. To quantify the necessary inter-channel conversion factors, I performed linear regression, using an ordinary least squares method, on Gaussian-filtered pixel values pooled from multiple untagged embryos. Then, to perform correction on images containing fluorophore, we just need to capture an AF channel image, alongside the GFP channel image, rescale the AF image according to the predefined conversion factors, and then subtract this inferred autofluorescence away from the GFP channel image pixel-by-pixel.\\


\subsubsection{Assessing SAIBR performance on images of PAR proteins}

To assess the effectiveness of SAIBR, and it’s utility in the analysis of PAR proteins, I applied it to a range of images of unlabelled and GFP-labelled embryos. As expected, applying SAIBR to images of unlabelled cells reduced fluorescence from across the cells to zero, with no visible structures remaining (\cref{fig:saibr_spatial_correction}A). This is a good validation of the method, and suggests that it can properly account for all of the autofluorescence in the cell. \\

In the case of LGL-1::GFP expressing embryos, where autofluorescence signal is dominant (\cref{fig:saibr_n2_vs_lgl}), SAIBR removes autofluorescence signal within the cell, and improves contrast at the posterior cortex, allowing us to better resolve plasma membrane enrichment \cref{fig:saibr_spatial_correction}B. Improvements are just as striking for PAR-3::GFP. In addition to improvements at the cortex, we see that SAIBR can suppress the local fluorescence minimum at the cell centre caused by lower AF at the pronuclei. For PAR-6::GFP the improvements are qualitatively less striking, as the ratio of fluorophore signal to autofluorescence is higher, but nonetheless AF removal has a quantitative impact.\\

\begin{figure}
\includegraphics[scale=0.9]{saibr_spatial_correction}
\centering
\mycaption{Applying SAIBR to images of GFP-tagged PAR proteins}{
\textbf{(A) - (D)} Raw (top) and SAIBR-corrected (middle) midplane images of zygotes expressing the indicated GFP fusions. Bottom panels show associated linescans across the images. SAIBR reveals prominent membrane localisation for LGL-1 and PAR-3 (arrows) which is not obvious in uncorrected images. We also see a prominent signal dip in the pronuclear region of uncorrected images due to local exclusion of autofluorescence (arrowheads), which disappears in SAIBR corrected images.
Fig. 2A-D from \textcite{Rodrigues2022}.}
\label{fig:saibr_spatial_correction}
\end{figure}

As shown in \cref{fig:saibr_membrane_profiles}, SAIBR has a strong impact on the shape of intensity profiles taken across the cortex within each polarity domain, in all cases showing a clearer peak and suppression of signal at the internal portion of the curves. This has particular importance for quantitative studies as, as described in section \ref{section:memquant}, the shape of cross-cortex profiles are often used to quantitatively analyse membrane concentrations and/or membrane affinities. For example, a cross-cortex profile with a central peak that is much higher than the internal cytoplasmic plateau clearly implies that the protein is binding to the membrane with a high affinity. We can see from the SAIBR corrected profiles that, of the three proteins shown here, LGL-1 has the strongest membrane affinity (highest enrichment at the membrane compared to its cytoplasmic level), followed by PAR-6, followed by PAR-3. If we look at the profiles pre-correction, however, this isn't so clearly apparent. One might have had some success by simply subtracting an equivalent average profile taken from untagged N2s, but such a method would fail to account for the fact that much of variation between embryos is down to autofluorescence, and would therefore be unsuitable for studies where inter-embryo variation is important. In the case of LGL-1 this would also clearly result in negative values at the cytoplasmic portion of the curve for some embryos.\\

\begin{figure}
\includegraphics[scale=1]{saibr_membrane_profiles}
\centering
\mycaption{SAIBR improves detection of membrane signals}{
\textbf{(A) - (C)} Averaged membrane profiles taken from raw (top) and SAIBR-corrected (bottom) images for LGL-1::GFP (A), PAR-3::GFP(B) and PAR-6::GFP (C) shown relative to unlabelled N2 controls. Membrane position at x=0$\mu m$. Mean $\pm$ SD indicated.
Fig. 2E-G from \textcite{Rodrigues2022}.}
\label{fig:saibr_membrane_profiles}
\end{figure}

\subsubsection{Extending SAIBR to dual-labelled \textit{C. elegans} embryos}

As SAIBR relies on a red shifted emission channel, complications can arise in samples containing red fluorophore. As red fluorophores are usually weakly excited by blue lasers, they will contribute additional signal to the AF channel, which may lead to overestimation, and therefore oversubtraction, of autofluorescence if not accounted for. If RFP levels are low, this effect may be small and can be ignored. However, if RFP levels are high, this bleedthrough effect can be significant. This can be demonstrated by observing the inter-channel relationship in control embryos expressing mCh::MEX-5 (\cref{fig:saibr_3channel_correlation}A). We find that, when an RFP is present, this relationship deviates significantly from the typical relationship observed in N2s, in direct proportion to local signal in the RFP (561nm excitation, 630/75nm emission) channel (\cref{fig:saibr_3channel_correlation}A inset). As this relationship is linear, autofluorescence in the GFP channel can be described as a linear function of both the AF and the RFP channels. Plotting pixel data in three dimensions shows that the data can be successfully fit to a plane, by performing multiple linear regression (\cref{fig:saibr_3channel_correlation}B, C). \\


\begin{figure}
\includegraphics[scale=1]{saibr_3channel_correlation}
\centering
\mycaption{Pixel intensities correlate across three channels in samples containing red fluorophore.}{
\textbf{(A)} Gaussian-filtered (radius = 1) GFP channel vs AF channel pixel intensities for an mCh::MEX-5 expressing embryo, colour coded by RFP-channel fluorescence. Dashed line indicates the AF vs GFP correlation for untagged (N2) embryos. Inset shows residuals as a function of RFP channel signal. Pixels taken from an ROI encompassing the embryo and a region of the background, and 10\% of pixels are shown at random.
\textbf{(B)} Multiple regression fit of GFP-channel signal as a function of AF and RFP channel signal. The same embryo and sample of pixels as in (A) is shown.
\textbf{(C)} Predicted vs. measured GFP-channel signal based on the fit in (B). The same embryo and sample of pixels as in (A) and (B) is shown.
Fig. 4B-D from \textcite{Rodrigues2022}.
}
\label{fig:saibr_3channel_correlation}
\end{figure}

To perform correction on images containing red fluorophore, we just need to capture all three channels, calculate autofluorescence using the three-channel regression relationship obtained from the appropriate RFP-tagged single line, and then subtract this away from the GFP channel image. This is demonstrated in \cref{fig:saibr_3channel_correction}, for embryos expressing both PAR-6::GFP and mCh::MEX-5, or just mCh::MEX-5. Whereas 2-channel SAIBR results in oversubtraction of autofluorescence (particularly visible in the mCh::MEX-5 single line), this is eliminated when using 3-channel SAIBR.\\

\begin{SCfigure}
\includegraphics[scale=1]{saibr_3channel_correction}
\mycaption{3-channel SAIBR prevents oversubtraction in samples containing red fluorophore}{
RFP channel, 2-channel SAIBR and 3-channel SAIBR images for samples expressing mCh::MEX-5, along with associated linescans. 2-channel SAIBR results in oversubtraction which is clearly visible as negative values in the mCh::MEX-5 line (arrow) and reduced cytoplasmic signal in the two-colour line. 3-channel SAIBR, using the calibration procedure shown in \cref{fig:saibr_3channel_correlation}, prevents oversubtraction.
Fig. 4F from \textcite{Rodrigues2022}.}
\label{fig:saibr_3channel_correction}
\end{SCfigure}


\subsubsection{Discussion}

In summary, I have demonstrated that a simple protocol, which we’ve termed SAIBR, can be used to successfully remove autofluorescence in images of \textit{C. elegans} zygotes. The improvements are particularly striking for images of fusion proteins with low levels of expression, such as LGL-1::GFP, but even in cases where expression levels are higher, AF correction will prove important for quantitative analysis of membrane concentrations, as discussed in the next section. For the rest of this thesis, unless otherwise stated, green fluorophore images will be presented with autofluorescence removed by SAIBR.\\

The simplicity of the method means that it can be easily incorporated into existing workflows, and should be applicable to a variety of imaging platforms. In the full study \citep{Rodrigues2022}, we show that the method is equally successful on both spinning-disk confocal and wide-field instruments. \\

Whilst initially developed with \textit{C. elegans} embryos in mind, the method isn't specific to this system, and could be applied to a number of other model systems in which autofluorescence is a problem. In the full study, we show that the method works successfully in \textit{C. elegans} larvae, as well as other model organisms such as starfish and yeast. That said, the method isn’t guaranteed to perform well in all cases. If samples contain multiple, independently varying sources of autoflourescence, then SAIBR may face problems as a single autofluorescence reporter channel cannot account for this. However, much like how we can tackle red fluorophores, we have found that in some cases this can be solved simply by adding one extra reporter channel. Inevitably, though, such an approach may not be compatible with dual-colour imaging. \\

Whilst the analysis steps are relatively straightforward, implementing the computational workflow may still be a barrier to adoption for some. Therefore, to make the protocol accessible, I have put together a simple GUI-based FIJI plugin which can carry out all the necessary analysis in a few simple steps. This can be found here: \url{https://github.com/tsmbland/saibr_fiji_plugin}. \\

The method comes with a few tradeoffs, which will vary in significance depending on the particular study. One issue is that, as the method combines pixel noise from multiple images, corrected images can in some cases be quite noisy, particularly where weak imaging conditions are used. It also requires capturing two emission channels for each image, which doubles sample illumination times and potential phototoxicity, which may be an issue for long timelapses. Additionally, if samples display rapid motion, then the time lag between taking these two channels may lead to pixel mismatches, which could introduce artefacts. These last points could be fixed by using an imaging setup that allows for dual capture of multiple emission bands. However, for this particular study, these issues will not be of major significance. \\


\clearpage
\subsection{Extracting membrane and cytoplasmic signal components}
\label{section:memquant}

Transition to section

\subsubsection{An overview of existing methods}

A number of methods have been implemented aiming to quantify protein concentrations at the membrane and cytoplasm in \textit{C. elegans} embryos. A typical approach to quantify membrane concentrations is to find the region of the image representing the cortex (either by manual or computational segmentation) and take a coarse measure of pixel values within this region (\cref{fig:memquant_method_comparison}, \textit{Intensity based quantification}). Such an approach was used by \textcite{Goehring2011a}, who manually segmented the embryo cortex, computationally straightened a region around the circumference of the cell, and defined membrane concentrations as the sum of the highest intensity group of pixels at each cortical cross section. \textcite{Hubatsch2019a} used a similar approach, but replaced manual segmentation with an automated computational pipeline. Similarly, \textcite{Zhang2017} developed a computational protocol to segment images, and defined membrane concentrations around the cell as the average signal intensity within a region representing the cell cortex.\\

\begin{figure}
\includegraphics[scale=1]{memquant_method_comparison}
\centering
\mycaption{Approaches for quantifying membrane concentrations from midplane images}{Typical methods involve segmentation of embryos, either manually or computationally, followed by computational straightening, to simplify geometry, here shown for an image of mNG::PAR-2. Each position along the x-dimension of the straightened image represents a profile perpendicular to and centred on the membrane at that position. A number of approaches can be used to extract quantitative information from these profiles. Intensity-based methods use pixel values within a central region of the profile representing the membrane (red) as a measure of membrane concentrations. Fitting-based methods fit the shape of the whole profile to a model, and extract relevant parameters from the fitted model. This is shown here for a model based on \textcite{Gross2018}, which describes total signal as a sum of a Gaussian component representing membrane signal and an error-function component representing cytoplasmic signal. Each component is characterised by an amplitude ($M$ and $C$) and a shared width parameter ($\sigma$).
}
\label{fig:memquant_method_comparison}
\end{figure}

A main disadvantage of these methods is that, as the membrane is immediately apposed to the underlying cytoplasm, pixel values at the membrane will inevitable contain a contribution from cytoplasmic fluorophore signal (and, indeed, cytoplasmic autofluorescence if this isn't accounted for). This means that measurements of membrane concentrations will be sensitive to changes in cytoplasmic concentrations, and means that these methods fail to achieve an accurate zero (a positive signal will always register, even if there is nothing on the membrane). Typically, attempts are made to overcome this latter point by normalising concentrations and/or subtracting away a local or global estimate of the background signal, but this is often difficult and inaccurate.\\

More advanced methods have aimed to overcome this problem by building models to describe the expected shape of individual cross-cortex profiles, based on summed contributions of cytoplasmic and membrane signal. Membrane and cytoplasmic concentrations can then be extracted by fitting measured profiles to this model, and extracting the relevant parameters describing the amplitudes of the two signal components (\cref{fig:memquant_method_comparison}, \textit{Fitting based quantification}). Such an approach was used by \textcite{Gross2018}, who described the cross-cortex profile at each point around the circumference of the embryo as the sum of a Gaussian and an error-function contribution, representing the expected form of a point (membrane) or step-function (cytoplasm) convolved with a Gaussian-like point spread function in 1D. A similar approach was previously used in \textcite{Blanchoud2015} (although with a different description of cytoplasmic signal).\\

\subsubsection{Accounting for out-of-focus scatter}

Whilst these methods have been effectively deployed in various studies, and are good at capturing a proper zero baseline, their accuracy is inevitably limited by the accuracy of the underlying models. For many imaging set-ups, the assumption that cytoplasmic and membrane contributions can be described by such simple mathematical functions may in fact be far from the truth. This was demonstrated by for \textit{C. elegans} embryos by \textcite{Reich2019a}, who quantitatively analysed cross-cortex signal in cells containing only cytoplasmic signal, finding that (under imaging conditions similar to those used in this study) the shape of this profile deviates significantly from the expected error-function shape. I have also performed similar analysis here  (\cref{fig:memquant_cyt_profile}), in this case using a line expressing free NeonGreen which is not expected to bind to the membrane, and using SAIBR to remove any autofluorescence contributions. Here we can see the true shape of the cytoplasmic signal contribution, which varies little within embryos (\cref{fig:memquant_cyt_profile}C) and between embryos (\cref{fig:memquant_cyt_profile}D).\\

\begin{figure}
\includegraphics[scale=1]{memquant_cyt_profile}
\centering
\mycaption{Assessing the contribution of cytoplasmic protein to midplane signal}{
\textbf{(A)} Midplane image of an embryo expressing free NeonGreen.
\textbf{(B)} Straightened cortex of the image in (A).
\textbf{(C)} Straightened image in (B), averaged over the x dimension. Mean $\pm$ SD.
\textbf{(D)} Mean ($\pm$ SD) cross-cortex profile, averaged across multiple embryos.
}
\label{fig:memquant_cyt_profile}
\end{figure}

The main reason that this shape deviates from an error-function is likely due to scattering and diffraction of light from planes above and below the imaging plane, combined with a curved geometry in the z-dimension (\cref{fig:memquant_cyt_psf}). Scatter, which is a common issue in images of biological samples, causes a broadening of light in three dimensions as it passes through regions of heterogeneous refractive index. This occurs within the (xy) plane of an image, but is typically far more significant in the z-axis. Whilst confocal microscopes are designed to only capture light from a single plane, the whole sample is illuminated, so they will capture any emitted light from other planes that scatters into the focal plane. This means that pixel intensities within the focal plane will be affected not only by structures within that plane but also structures above and below.\\


\begin{figure}
\includegraphics[scale=0.9]{memquant_cyt_psf}
\centering
\mycaption{Effects of sample geometry and the PSF on in-plane signal measurements in simulations of cytoplasm-like signal}{
\textbf{(A) - (C)} (C) shows simulated midplane signals derived from the geometry shown in (A) convolved with three different point spread functions (B).
\textbf{(D) - (F)} (F) shows simulated midplane signals derived from three geometries shown in (D) convolved with the point spread function shown in (E). Geometries are designed to resemble cytoplasmic protein at the edge of a curved cell. Red dashed lines in (A) and (D) indicate midplane. PSFs modelled as 2D Gaussians, which is not representative of real PSFs.
}
\label{fig:memquant_cyt_psf}
\end{figure}

It is likely that a similar problem also applies in the case of membrane protein (\cref{fig:memquant_mem_psf}). Specifically, this analysis shows that out of focus membrane signal might be expected to lead to a shape resembling an asymmetric Gaussian, with higher signal at the interior of the cell than the exterior. In fact, this phenomenon can be observed just by looking at images of polarised PAR proteins (e.g. \cref{fig:memquant_method_comparison} for PAR-2), where out-of-focus membrane signal can create the appearance of a cytoplasmic gradient. By comparison, two-photon images of PAR-2, which aim to eliminate out of focus excitation, show a completely flat cytoplasm \citep{Petrasek2008}, which is expected for most PAR proteins based on fast measured diffusion rates (PAR-3 and PAR-1 excepting). Whilst in some cases this out-of-focus bleedthrough may be of little concern, it might be particularly problematic if accurate cytoplasmic quantification is required, as this out of focus signal may be attributed to cytoplasmic protein. Without accurate cytoplasmic concentration, measures of membrane to cytoplasmic ratios, which are often used as a read-out of membrane affinity, may be wildly off. This will prove significant for much of the analysis in later chapters of this thesis.\\

\begin{figure}
\includegraphics[scale=0.9]{memquant_mem_psf}
\centering
\mycaption{Effects of sample geometry and the PSF on in-plane signal measurements in simulations of membrane-like signal}{
\textbf{(A) - (C)} (C) shows simulated midplane signals derived from the geometry shown in (A) convolved with three different point spread functions (B).
\textbf{(D) - (F)} (F) shows simulated midplane signals derived from three geometries shown in (D) convolved with the point spread function shown in (E). Geometries are designed to resemble membrane protein at the edge of a curved cell. Red dashed lines in (A) and (D) indicate midplane. PSFs modelled as 2D Gaussians, which is not representative of real PSFs.
}
\label{fig:memquant_mem_psf}
\end{figure}

Thus, if we want to obtain accurate measures of cytoplasmic and membrane protein concentrations, we need a method that can account for out-of-focus scatter. One common approach to account for out-of-focus scatter is to take a z-stack across the whole sample and apply a deconvolution algorithm to the 3D stack to reassign all blurred/scattered light to an in-focus location \citep{Wallace2001}. These methods rely on prior knowledge of the point spread function (PSF) that applies to the particular sample and imaging set-up, which needs to be as accurate as possible, otherwise artefacts can arise. Theoretical methods exist to estimate an appropriate PSF given parameters such as the imaging modality, numerical aperture and emitted light wavelength. However, whilst these methods are good at describing blur within the imaging apparatus, scattering within the sample and at the sample-apparatus interface is difficult to model accurately. For this reason, it can be more effective to measure an empirical PSF by imaging the 3D light distribution from a single point source (e.g. a fluorescent bead) under similar sample prep conditions to the sample of interest. However, as PSFs are influenced by scatter within the sample itself, the accuracy of this method depends on how closely the sample environment can be replicated when imaging the beads, which is not trivial. \\

Furthermore, most deconvolution methods assume that the PSF is a constant function throughout the whole image, but in many cases this won't be the case. There may, for example, be refractive index gradients within the sample, which will alter the shape of the PSF depending on location within the sample. Additionally, if there is a mismatch between the refractive index of the immersion and mounting media, as is often unavoidable when imaging live biological samples, then the PSF will usually vary with depth as spherical aberrations will be introduced deeper into the sample. A PSF from a fluorescent bead located directly below the coverslip will not capture either of these phenomena.\\

In reality, given all of these confounding factors, an accurate description of the PSF that applies to a given sample of interest is often unachievable in practice. Whilst deconvolution with a suboptimal PSF may be sufficient for many qualitative applications, accurate quantitative measurements cannot be guaranteed. For this reason, I opted against using a PSF-based deconvolution approach to account for out-of-plane scattering.\\

Fortunately, in this particular case, matters are simplified by the fact that the geometries of protein distribution are usually highly consistent. Not only is the shape of embryos highly consistent, but PAR protein distributions within the embryo also tend to display rotational symmetry (at least during normal polarity development in P0), meaning that protein distributions in planes above and below the focal plane tend to be similar/identical to those seen at the focal plane (much like the simulations in \cref{fig:memquant_cyt_psf} and \cref{fig:memquant_mem_psf}). Optical properties are also not expected to change from sample to sample, or from location to location around the circumference of an embryo.\\

Together, these features imply that, in cases where protein is either entirely cytoplasmic or entirely membrane-bound, the normalised shape of the cross-cortex profile measured at the midplane should be some constant function, that shouldn't vary much between embryos or spatially around the circumference of an embryo. A change in local or global concentrations should amount to a rescaling of these profiles, but shouldn't change their normalised shape. Where there is a mix of cytoplasmic and membrane protein, the total measured cross-cortex profile will then be a sum of these two contributions. Therefore, if one has prior knowledge of the expected shape of the cross-cortex profile for cytoplasmic-only and membrane-only protein, then measured profiles can be fit as the sum of these two contributions, and membrane/cytoplasmic concentrations extracted as the amplitudes of the two signal components.\\

Therefore, a key step in the path to accurate quantification is the ability to measure appropriate reference profiles describing the contributions of cytoplasmic and membrane protein. As mentioned previously, cytoplasmic reference profiles can easily be obtained by analysing cells in which all protein is cytoplasmic (\cref{fig:memquant_cyt_profile}). \textcite{Reich2019a} (and \textcite{Reich2019}) used an approach half-way between the \textcite{Gross2018} method and the method that I am proposing, replacing the error-function description of cytoplasmic signal with a measured cytoplasmic reference profile. Thus, total signals were fit as the sum of a Gaussian profile and this reference profile, which resulted in a better ability of the model to fit the shape of measured profiles.\\

Whilst this move is a significant step in the right direction, the problem remains of how best to account for out of focus membrane light. This presents a challenge: whilst it is relatively easy to directly measure a cytoplasmic reference profile (you just need a reference image in which all signal is cytoplasmic), the same is not true for a membrane profile as it is difficult to find a reference case in which all protein is membrane bound. To extend the method to account for out-of-focus membrane protein, I attempted to find alternative methods to get an approximation of the membrane reference profile. An initial idea, which involved staining the exterior of the embryo eggshell with a fluorescent dye, proved technically challenging and not reproducible.\\

Instead, the problem becomes simplified when one considers that, even in cases where the membrane is polarised, the cytoplasmic pool of most PAR proteins should be uniform (although exceptions do exist in the case of PAR-1 and PAR-3, where true cytoplasmic gradients have been observed). Thus, straightened cortices can be modelled as a uniform cytoplasmic component, defined by a cytoplasmic reference profile ($R_c$) and a single, uniform, cytoplasmic concentration ($C$), plus a polarised membrane component, defined by a membrane reference profile ($R_m$) and a nonuniform membrane concentration profile ($M$), as shown in \cref{fig:memquant_model_schematic}. This is essentially a generalisation of the \textit{Fitting based quantification} method presented in \cref{fig:memquant_method_comparison}, but proposes the use of arbitrary membrane and cytoplasmic reference profiles rather than mathematical functions (e.g. Gaussian, error function), and has the restriction of a uniform cytoplasm.\\

\begin{figure}
\includegraphics[scale=1.15]{memquant_model_schematic}
\centering
\mycaption{A model for simulating membrane and cytoplasmic fluorescence signal at the cortex}{
Simulated images of straightened cortices are described as the sum of membrane and cytoplasmic signal components. The membrane component is described as a 1D array of amplitudes ($M$), representing concentrations around the circumference of the embryo, convolved with a membrane reference profile ($R_m$). The cytoplasmic component is described as a single uniform amplitude ($C$) convolved with a cytoplasmic reference profile ($R_c$). Blur/scatter in the x dimension is ignored as adjacent positions are assumed to be similar.
}
\label{fig:memquant_model_schematic}
\end{figure}

$R_c$ can easily be predefined by imaging embryos in which all protein is cytoplasmic (\cref{fig:memquant_cyt_profile}). If we assume for now that $R_m$ is also predefined, then $M$ and $C$ for a given embryo can be determined by fitting a straightened image to the model presented in \cref{fig:memquant_model_schematic}. (At this point concentrations will be in arbitrary units, but I'll return to this point later). Whilst $R_m$ is in fact not predefined, given the constraints imposed by cytoplasmic uniformity, only a model with an appropriate $R_m$ will be able to create simulated images that closely match experimental images. (Imagine, for example a model with a Gaussian membrane reference profile, which would clearly fail to capture graded interior signal). Therefore, under conditions such as these, in which we have a uniform cytoplasmic component and a graded membrane component, $R_m$ need not be predefined, and can simply be fit to the data along with the concentration parameters. To perform this kind of optimisation, I have developed a gradient descent approach based on differentiable programming, which I describe below.\\


\subsubsection{A gradient descent protocol for image quantification}

Gradient descent is a popular optimisation strategy used for a number of machine learning applications. When optimising a model described by a series of parameters, the idea of gradient descent is to calculate the partial derivative of each parameter with respect to a loss term (e.g. mean squared error). A negative gradient for a certain parameter would imply that an increase in that parameter would decrease the loss term, whereas a positive gradient would imply the opposite. Therefore, to reduce loss, each input parameter can be adjusted in proportion to the negative of its partial derivative. Starting with a set of initial conditions, this procedure is iteratively repeated, adjusting parameters and calculating new gradients at each step, until the loss term reaches a minimum. \\

The utility of these methods has been greatly advanced in recent years by the development of differentiable programming tools. Commonly used for deep learning, although generalisable to other problems, these tools greatly speed up computation for complex optimisation procedures by automatically calculating gradients, rather than relying on numerical methods, using a process known as backpropagation. In addition, extensions to the basic gradient descent algorithm have proven effective at speeding up convergence and preventing entrapment in local minima \citep{Sun2019}. \\

In the case of this particular problem, the procedure is described in \cref{fig:memquant_forward_and_back_propagation}. Given a set of parameters ($M$, $C$, $R_m$, $R_c$), a forward propagation step simulates an image, and this is compared to the ground truth image to calculate a loss term. Backpropagation then calculates the gradient of each of the input parameters with respect to this loss term. At this point, we can adjust some or all of the input parameters according to these gradients. Repeating this cycle of forward and back propagation will then lead to a gradual optimisation of these parameters, until the loss function reaches a minimum. \\

\begin{figure}
\includegraphics[scale=1.1]{memquant_forward_and_back_propagation}
\centering
\mycaption{Backpropagation gradient descent protocol for parameter optimisation}{Parameters are optimised by an iterative cycle of forward and back propagation. In the forward propagation step (top), an image is simulated given input parameters $M$, $C$, $R_m$, $R_c$, and compared to the ground truth image to calculate a loss term. In the back propagation step (bottom) gradients of the loss term with respect to each of the input parameters are calculated using the chain rule method. A subset of the input parameters can then be changed according to these gradients, which will depend on the procedure (e.g. \cref{fig:memquant_membg_training} vs \cref{fig:memquant_benchmarking_ph_rundown}). This process is repeated many times until optimisation reaches a plateau. Dimensions of the variables are shown in blue, where p represents the number of positions to fit around the cortex, and t represents the thickness of the straightened image.}
\label{fig:memquant_forward_and_back_propagation}
\end{figure}

To test this approach, I built a model using the differentiable programming package Tensorflow \citep{Abadi2016}, and first applied it to images of polarised PAR-2. The model was initiated with all concentrations ($M$ and $C$) equal to zero, and $R_m$ initiated as a Gaussian. For $R_c$ I used a measured profile (\cref{fig:memquant_cyt_profile}), and this was not adjusted during training (\cref{fig:memquant_membg_training}A). Using an Adam optimiser \citep{Kingma2015} with a learning rate of 0.01, all other parameters ($M$, $C$ and $R_m$) were then adjusted iteratively until a plateau was reached (250 steps), as shown in \cref{fig:memquant_membg_training}B. The final simulated image, composed of a uniform cytoplasmic component and a nonuniform membrane component, closely matches the ground truth image (\cref{fig:memquant_membg_training}C).\\


\begin{figure}
\includegraphics[scale=1]{memquant_membg_training}
\centering
\mycaption{Separation of membrane and cytoplasmic signal in an image of mNG::PAR-2}{
\textbf{(A)} Schematic demonstrating the optimisation procedure, highlighting the input parameters to be optimised in red.
\textbf{(B)} $R_m$, $M$ and $C$ throughout the training procedure for an mNG::PAR-2 expressing polarised embryo. $R_m$ was initiated as a Gaussian (radius = x) centred at zero, $M$ and $C$ initiated as zero.
\textbf{(C)} Final simulated image compared to the ground truth image. Right panel shows profiles at specified positions (solid = ground truth, dashed = simulated). Uniform cytoplasmic component shared between profiles indicated in gray.
}
\label{fig:memquant_membg_training}
\end{figure}

% more here on reproducibility.

\subsubsection{Segmentation}

In addition to the parameters already mentioned, the full model also includes a series of alignment parameters, which can also be trained by gradient descent, allowing the model to freely align to the data in the y direction. As a result, ground truth images do not need to be accurately segmented prior to optimisation, and rough manual ROIs are fine to use. This is particularly useful for timelapse movies, where, even if the embryo undergoes shape changes, only a single manual ROI needs to be provided. An additional outcome of alignment is that the offset parameters can be used to refine the original ROI, meaning that the method can serve as a tool for computational segmentation. Refined ROIs can then be used to re-straighten the cortex, and optimisation repeated (\cref{fig:memquant_segmentation}). Overall, this allows cortices to be segmented with subpixel accuracy with minimal manual input. In theory, the initial manual step could also be automated using deep learning methods to create a fully automated pipeline \citep{Minaee2021}.\\

\begin{figure}
\includegraphics[scale=1]{memquant_segmentation}
\centering
\mycaption{Semi-automated segmentation algorithm}{
Segmentation is initiated with a rough manual ROI. The cortex is then straightened and fit to the model yielding, among other parameters, parameters describing offset at each position along the x dimension. These offsets can then be used to adjust the initial ROI, and re-straighten the cortex. The re-straightened cortex can then be refit to the model to give concentration parameters. Shown here for an embryo expressing mNG::PAR-2(L109R).
}
\label{fig:memquant_segmentation}
\end{figure}


\subsubsection{Benchmarking the method}

The method described so far is limited to cases of images with polarised membranes. However, as discussed previously, $R_m$, which is a function of local geometry and optical properties, should be a constant function applicable to all embryos. Therefore, much like I used a predefined $R_c$ when fitting images of polarised PAR-2, images of proteins without a polarised membrane can be quantified by using an $R_m$ derived from a calibration procedure on polarised images. \\

To test this method, I performed quantification on images of embryos expressing the uniform plasma membrane probe GFP::PH with variable expression levels (\cref{fig:memquant_benchmarking_ph_rundown}B), obtained by performing an RNAi rundown using XFP RNAi feeding bacteria (see Methods). In this case, I used a predefined $R_m$ and $R_c$, and only optimised $M$ and $C$ (as well as the alignment parameters described in the previous section) (\cref{fig:memquant_benchmarking_ph_rundown}A). Images were initiated with a rough manual ROI and segmented using the method described in \cref{fig:memquant_segmentation} prior to final quantification. Compatible with expected linear membrane binding kinetics, we can see that the method gives a tight linear relationship between cytoplasmic and membrane concentrations (\cref{fig:memquant_benchmarking_ph_rundown}C). N2s are also accurately described as having cytoplasmic and membrane concentrations close to zero.\\

\begin{figure}
\includegraphics[scale=1]{memquant_benchmarking_ph_rundown}
\centering
\mycaption{Quantification of GFP::PH expressing embryos reveals a linear relationship between membrane and cytoplasmic concentrations}{
\textbf{(A)} Schematic demonstrating the optimisation procedure, highlighting the input parameters to be optimised in red.
\textbf{(B)} Images of GFP::PH expressing embryos with varying dosages of GFP::PH, obtained by RNAi rundown.
\textbf{(C)} Membrane vs. cytoplasmic concentrations for the full dataset of embryos. Empty circles represent untagged N2 embryos. Embryos in (B) represented by blue, orange and green circles.
\textbf{(D)} Cross-cortex profiles averaged over the entire circumference of the embryo vs. GFP::PH dosage. Colour coding as in (C).
}
\label{fig:memquant_benchmarking_ph_rundown}
\end{figure}

I next investigated how robust the pipeline is to changing signal-to-noise ratios, using images of three GFP::PH embryos with varying expression levels and adding varying levels of Gaussian pixel noise (\cref{fig:memquant_benchmarking_noise}A). As seen in \cref{fig:memquant_benchmarking_noise}B and C, pixel noise adds noise to the resulting quantifications, but doesn't bias the data in any direction. Quantification of untagged N2s is also not biased by noise (\cref{fig:memquant_benchmarking_noise}B/C grey points).\\

\begin{figure}
\includegraphics[scale=0.95]{memquant_benchmarking_noise}
\centering
\mycaption{Quantifications of membrane and cytoplasmic concentrations are robust to pixel-noise}{
\textbf{(A)} Images of three GFP::PH expressing embryos with varying amounts of GFP::PH subject to varying levels of Gaussian pixel noise.
\textbf{(B)} Quantifications of membrane concentration (averaged over the entire embryo) for the three embryos in (A) subject to varying levels of pixel noise, colour coded by embryo. For each noise level 10 images were generated for each embryo, and each images was quantified and plot as a single point. Similar analysis for an untagged N2 embryo is also shown (gray). Dashed lines indicate quantification at noise=0.
\textbf{(C)} Quantifications of cytoplasmic concentration for the three embryos in (A) and a single N2 embryo subject to varying levels of pixel noise. Procedure as in (B). Noise level defined as standard deviation of Gaussian noise added to the image. 
}
\label{fig:memquant_benchmarking_noise}
\end{figure}


\subsubsection{Calibrating concentration units}

As $M$ and $C$ are in arbitrary units (effectively in units of their own respective reference profiles), a conversion parameter, is required to put them into common units. To calibrate this conversion parameter, I quantified the effects on $M$ and $C$ measurements of redistributing a fixed pool of protein from the cytoplasm to the membrane. To do this, I used an optogenetics system with a plasma membrane bound PH::eGFP::LOV to move a cytoplasmic pool of ePDZ::mCherry to the membrane (Fielmich et al., 2018). Embryos were exposed to blue light for 10 seconds, which promotes binding between ePDZ and LOV, leading to a rapid uniform recruitment of ePDZ::mCherry to the membrane and a reduction in the concentration in the cytoplasm (\cref{fig:memquant_optogenetics}). Where ePDZ::mcherry is expressed alone, this localisation shift isn't observed (\cref{fig:memquant_optogenetics}, grey points).\\

\begin{figure}
\includegraphics[scale=1]{memquant_optogenetics}
\centering
\mycaption{Redistribution of fluorescent signal from cytoplasm to membrane using optogenetics}{
\textbf{(A)} ePDZ::mCherry before (top) and after (bottom) exposure to blue light, in an embryo also expressing PH::eGFP::LOV.
\textbf{(B)} Cytoplasmic concentration of ePDZ::mCherry (in arbitrary units) before and after blue light exposure for embryos with (blue) and without (gray) PH::eGFP::LOV. 
\textbf{(C)} Membrane concentration of ePDZ::mCherry (in arbitrary units) before and after blue light exposure for embryos with (blue) and without (gray) PH::eGFP::LOV. 
}
\label{fig:memquant_optogenetics}
\end{figure}

Whilst there is significant protein relocalisation in the optogenetic system, the total amount of protein before and after blue light exposure can be assumed to be constant. This total amount, $T$, can be expressed as the $C$ value that would be expected if all tagged molecules were in the cytoplasm, given by equation \ref{eq:t} where $\psi$ is the surface-area to volume ratio of the cell ($= 0.174 \mu m ^ {-1}$ \citep{Goehring2011a}). Given that $M$ is in different arbitrary units to $C$, a conversion parameter, $c$, is required:

\begin{equation}
T = C + \psi c M
\label{eq:t}
\end{equation}

Given that $T$ is the same before and after exposure, $c$ can be calculated, on an embryo by embryo basis, by comparing the gain in $M$ post-exposure to the loss in $C$:

\begin{equation}
c = \frac{C_{pre \textrm{-} exposure} - C_{post  \textrm{-} exposure}}{\psi (M_{post  \textrm{-} exposure} - M_{pre  \textrm{-} exposure})}
\end{equation}\\

Performing this analysis on the dataset in \cref{fig:memquant_optogenetics} (blue points) gives a value of $c$ = 2.88 $\pm$ 0.12 $\mu$m (mean $\pm$ SD), which can be used to convert membrane concentrations to the same common units as cytoplasmic concentrations (i.e. $\mu m^{-3}$ for cytoplasmic concentrations and $\mu m^{-2}$ for membrane concentrations). Note that these concentrations are still arbitrary in the sense that they do not indicate absolute concentrations (i.e. absolute number of molecules per unit area). However, for much of the analysis in the following sections, where the aim is to measure membrane affinities (membrane to cytoplasmic ratios) this will not be an issue, although I'll return to this point in section x.\\

\clearpage
\subsubsection{Discussion}

Accurate quantification of features from images relies on the ability to separate overlapping signals and correctly attribute signals to their source. In this section, I have described a two-step pipeline designed for accurate quantification of cytoplasmic and membrane concentrations from midplane images of \textit{C. elegans} zygotes. The first step involves separation of autofluorescence and fluorophore signal, and the second step involves separation of signals from cytoplasmic and membrane protein. The overall pipeline is not specific for any particular microscope, and makes no assumptions about the spectral characteristics of the signal components or the optical properties of the imaging system/sample. Whilst the SAIBR method isn't fundamentally tied to \textit{C. elegans}, and has been shown to apply to other systems, the method for separation of cytoplasmic and membrane signals is less generalisable, and a number of assumptions in the model presented here are firmly linked to the simple and reproducible geometries of \textit{C. elegans} zygotes and PAR protein patterns. Nevertheless, the ability to confidently quantify relative membrane and cytoplasmic concentrations in vivo brings forward new experimental possibilities for studies of the \textit{C. elegans} PAR network, and will prove fundamental to much of the work presented in the following chapters of this thesis.\\


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\section{Identifying core patterning behaviours of PAR-2}

\subsection{Introduction}

% Note: previous studies have shown that PAR-2 can maintain domains once formed, but hasn't been demonstrated whether PAR-2 can break symmetry without clearance of aPARs.

\subsection{Self-organisation of PAR-2 drives polarity in a uniform aPAR field}

\begin{figure}[!h]
\includegraphics[scale=0.9]{uniform_polarity}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:uniform_polarity}
\end{figure}

\begin{figure}[!h]
\includegraphics[scale=0.9]{uniform_polarity_sb}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:uniform_polarity_sb}
\end{figure}

\clearpage
\subsection{PAR-2 driven polarity is disrupted in various PAR-2 mutants}

\begin{figure}[!h]
\includegraphics[scale=0.9]{uniform_polarity_mutants}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:uniform_polarity_mutants}
\end{figure}

\clearpage
\subsection{Quantitative characterisation of PAR-2 mutants}

\begin{figure}
\includegraphics[scale=0.95]{par2_misc_mutants}
\centering
\mycaption{PAR-2 wild type and mutant alleles in polarised and aPAR mutant systems}{
\textbf{(A)} Midplane confocal images of GFP::PAR-2 (wt) and mutant alleles (top) along with linescans (bottom) in polarised cells with intact aPAR.
\textbf{(B)} Midplane confocal images of GFP::PAR-2 (wt) and mutant alleles (top) along with linescans (bottom) in uniform cells lacking functional PAR-3. Linescans are averaged over a 20-pixel wide band across the centre of the images, as indicated.
}
\label{fig:par2_misc_mutants}
\end{figure}

\begin{figure}
\includegraphics[scale=0.95]{par2_misc_mutants_quantification}
\centering
\mycaption{Quantification of wild-type PAR-2 and mutant alleles in polarised and aPAR mutant systems}{
\textbf{(A) - (D)} Quantification of cytoplasmic concentration (A), posterior membrane concentration (B), posterior membrane to cytoplasmic ratio (C) and total expression (D) for wild type and mutant GFP::PAR-2 in polarised cells with intact aPAR. Posterior membrane concentrations are given as the average membrane concentration over the posterior-most 20\% of the embryo.
\textbf{(E) - (H)} Quantification of cytoplasmic concentration (E), membrane concentration (F), membrane to cytoplasmic ratio (G) and total expression (H) for wild type and mutant GFP::PAR-2 in uniform cells lacking functional PAR-3.  Membrane concentrations are given as the average membrane concentration over the entire cell.
}
\label{fig:par2_misc_mutants_quantification}
\end{figure}

\clearpage
\subsection{RING domain promotes positive feedback to achieve optimal membrane affinity}

Transition to section\\

To test for evidence of feedback, I performed an assay where I titrated the level of PAR-2 by RNAi (in a par-3 mutant system) and quantified the membrane and cytoplasmic concentrations in the different embryos (similar to the assay performed previously for PH). Whilst a protein without feedback is expected to follow a linear relationship, a protein feeding back on its own membrane recruitment may instead be expected to have a higher membrane affinity the more total protein there is in the system, giving a nonlinear relationship between membrane and cytoplasmic concentrations. In the case of PAR-2, the assay quite clearly shows that there isn't a straightforward linear relationship between membrane and cytoplasm, with a positive uptick that's characteristic of positive feedback (\cref{fig:rundown_vs_c56s}).\\

On the other hand, mutant PAR-2 (C56S), as well as following a lower affinity trajectory, appears to have a more linear relationship between cytoplasm and membrane, more in line with a protein that's exchanging on and off the membrane without feedback. This suggests that the mechanisms of positive feedback, which appear to contribute to the strong membrane binding kinetics of PAR-2, may be RING domain dependent.

\begin{figure}[!h]
\includegraphics[scale=1]{rundown_vs_c56s}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:rundown_vs_c56s}
\end{figure}

\clearpage
\subsection{Discussion}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\section{Determining mechanisms of PAR-2 RING domain action}

\subsection{Introduction}

Results in the previous chapter suggest that the RING domain of PAR-2 plays a key role in regulating the membrane affinity of the protein, and may be responsible for conferring nonlinear-membrane binding kinetics to PAR-2. However, the precise mechanisms of RING domain action in this context are not yet understood, and are subject to investigation in this chapter.\\

Whilst no mechanisms have been definitively attributed to the RING domain, several observations in previous studies provide insight into potential mechanisms. On its own, the PAR-2 RING domain has been shown to lack any membrane binding activity (\textcite{Hao2006}, and replicated later in \cref{fig:ring_fragment_in_vivo}), whereas a central portion of the protein containing the polybasic regions but lacking the RING domain binds to the membrane with a similar affinity to PAR-2 (C56S). Therefore, rather than a direct interaction with the membrane, it’s likely that the RING domain plays a modulatory role, affecting electrostatic membrane binding via the polybasic regions. Additionally, the authors show that the weakened affinity of PAR-2 (C56S) cannot be restored by the presence of wild-type PAR-2. I have also performed similar analysis in \cref{fig:c56s_cis}, which shows that PAR-2 (C56S) membrane affinity is unchanged when balanced by a wild type copy compared to when homozygous. In other words, the effects of C56S mutation appear to be cis-acting (intrinsic to each molecule). Finally, FRAP studies of PAR-2 (C56S) show that the mutant has faster membrane turnover kinetics than wild type PAR-2, suggesting that weakened membrane affinity in the mutant is due to a failure to stably associate with the membrane \citep{Motegi2011}.\\


\begin{figure}
\includegraphics[scale=1]{c56s_cis}
\centering
\mycaption{The PAR-2 RING domain activity is cis-acting}{
\textbf{(A)} Schematic of the three genotypes investigated. Quantification results represent the distribution of mNG-tagged PAR-2 only.
\textbf{(B)} Normalised cross-cortex profiles averaged over the posterior domain for the three conditions. 
\textbf{(C)} Membrane affinity measurements for the three conditions, measured over the posterior domain of polarised cells.
}
\label{fig:c56s_cis}
\end{figure}

RING domains were originally described by \textcite{Freemont1991} as a commonly occurring motif, between 40 and 70 residues in length, with a characteristic pattern of cysteine and/or histidine residues. Structural studies have since revealed a ‘cross-brace’ pattern, whereby these cysteine and histidine residues interact with two zinc ions to form a characteristic folded structure (\cref{fig:ring_schematic}). RING domains are highly prevalent, with 300 RING domain containing genes in the human genome \citep{Li2008}. Two roles have been commonly attributed to RING domains in other proteins: ubiquitination and dimerisation. In this chapter, I investigate each of these in turn as potential mechanism in the context of PAR-2 membrane association.\\

\begin{SCfigure}
\includegraphics[scale=0.4]{ring_schematic}
\mycaption{Schematic of RING domain organisation}{
Cysteines interacting with zinc are labelled as C1 etc. H1 represents a zinc-interacting histidine. Xn refers to the number of amino acids in the spacer regions between the zinc ligands. Figure from \textcite{Deshaies2009}.
}
\label{fig:ring_schematic}
\end{SCfigure}


\subsection{Exploring a role for ubiquitination}

The most commonly associated role of RING domains is in the ubiquitination pathway, the mechanism by which ubiquitin (Ub) molecules are attached to substrate proteins. This pathway involves three classes of enzymes which catalyse three steps (\cref{fig:ubiquitination_pathway}): ubiquitin-activation (E1), ubiquitin-conjugation (E2), ubiquitin-ligation (E3), the latter of which is commonly attributed to RING domain proteins. E1 activates ubiquitin and transfers to an E2, where a thioester bond is formed between the E2 and Ub. E3s then simultaneously interact with a substrate and the E2-Ub conjugate to catalyse the transfer of Ub to a target lysine on the substrate. \\

\begin{figure}[!h]
\includegraphics[scale=0.95]{ubiquitination_pathway}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Schematic of the ubiquitination pathway}{Figure from \cite{Deshaies2009}}
\label{fig:ubiquitination_pathway}
\end{figure}

A single E1 works with several E2s ($\sim$ 40 in humans, 22 in \textit{C. elegans}), which work with hundreds of E3s. The choice of E2 often determines the type of ubiquitination that occurs. This can be either monoubiquitination, where a single ubiquitin molecule is added to a target lysine, or polyubiquitination, meaning the formation of a ubiquitin chain via lysine molecules on ubiquitin monomers. The functional consequences of ubiquitination are directly related to the type of ubiquitination that occurs. Monoubiquitination is associated with modification of protein function, and can regulate the structure of a protein, change it's binding partners or change it's cellular localisation. Depending on the type of linkage, polyubiquitination can also be associated with modification of protein function, but is most commonly associated with proteasome-dependent proteolysis. Each E3 typically interacts with only a small number of E2s, and a small number of substrates. Therefore, by acting as a bridge between a specific E2 and a specific substrate, E3s are responsible for conferring specificity to the ubiquitination reaction. \\

It is thought that most RING domain proteins possess E3 ligase activity \citep{Deshaies2009}, catalysing the transfer of Ub directly from an E2 to a target substrate. RING E3s work by binding to and stabilising the closed conformation of an E2-Ub conjugate. This alters the geometry of the thioester bond between Ub and E2, increasing reactivity towards a substrate lysine residue. At the same time as interacting with the E2, most RING E3s bind to a substrate, bringing a specific lysine on the substrate into close proximity of the E2. This then leads to direct transfer of Ub from the E2 to the substrate. Typically, this E3/substrate interaction is via other parts of the E3 protein, rather than the RING domain itself (e.g. the TRIM class of E3s use C-terminal substrate recognition domains).\\

Alternatively, lysines on the E3 protein itself can be targeted (autoubiquitination). In many cases this may be a side effect of E3 activity with no functional role, but in some cases this may serve a purpose. For example, autoubiquitination of BCA2 has been demonstrated as an elegant mechanism to regulate steady state levels of the protein \citep{Amemiya2008}. As autoubiquitination is competitive with substrate ubiquitination, the protein is able to regulate its own levels according to substrate availability.\\

Given the common role for RING domain proteins as ubiquitin ligases, this function has long been suspected for PAR-2, but definitive evidence for ubiquitin ligase activity by PAR-2 has so far not been demonstrated. However, compatible with a potential role, yeast two hybrid assays show that PAR-2 interacts moderately with several E2 enzymes \citep{Gudgen2004}. Given that autoubiquitination is common for E3s, and that ubiquitination can change the properties of a protein, it's plausible that the defects observed in C56S mutants may relate to a failure to autoubiquitinate. In this section I aim to test PAR-2 for ubiquitination activity, and investigate the potential role of this in vivo.\\

\subsubsection{PAR-2 does not show activity in in vitro ubiquitination assays}

\textit{Work in this section was performed with Colin Davis in Anne Schreiber's lab, and Diego Esposito in Katrin Rittinger's lab, both at the Crick}\\

My first aim was to test for the ability of PAR-2 to act as a ubiquitin ligase. Typically, this property can be assayed in vitro using purified samples of a protein of interest. To express and purify PAR-2 we used an insect cell system, which was chosen for its ability to express high levels of large, complex proteins. We expressed strep-tagged full-length PAR-2 in insect cells, lysed cells by sonication, and purified by strep pull-down followed by size exclusion chromatography (SEC) (see Methods). From the initial pull-down stage, we observed two prominant bands close to the expected mass of PAR-2. Due to their similar masess, we were unable to separate the two bands by size exclusion chromatography (\cref{fig:ubiquitin_assay}A). We tested both bands by mass spectrometry, which confirmed them both to be PAR-2. Interestingly, a single site (K351) was found to be ubiquitinated in the sample corresponding to the upper band, explaining the higher mass of this population on the gel compared to the expected mass of strep-tagged PAR-2 (75.24 kDa). No ubiquitinated sites were found in the sample corresponding to the lower band, which sits closer to the expected weight of PAR-2, although the K351 site was not covered by in mass spectrometry results, so we cannot confirm that this site wasn't ubiquitinated. We also observed two similar bands in a small-scale prep of PAR-2 (C56S) (data not shown). Therefore, I do not believe this to be a result of autoubiquitination, but may rather represent ubiquitination of the protein by an insect ubiquitin ligase within the insect cells. CRISPR mutation of K351 to an alanine has no effect on PAR-2 localisation in vivo (data not shown), and previous attempts to purify PAR-2 from worm lysates and HEK cell preps have not shown two bands.\\

Sample from SEC purification was pooled (lanes x-x) and used for an in vitro ubiquitination assay. Designed to test for ubiquitination activity of a protein, this assay involves mixing a text protein with ubiquitin, E1 and an E2, and observing for autoubiquitination of the test protein (see Methods). Given that E3s typically work with only a small number of E2s, the assay is sensitive to the choice of E2, and therefore care must be taken to choose an appropriate E2. In the case of PAR-2 this is so far unknown, although PAR-2 has been shown to weakly interact with UBC-2, UBC-18 and UEV-2 in yeast two hybrid assays \citep{Gudgen2004}. Since UBC-2 (known as UBCH5a in Humans) is also the most promiscuous of the E2s, this was chosen for the assay. As shown in \cref{fig:ubiquitin_assay}, whereas a positive control sample (TRIM2) showed rapid auto-polyubiquitination and depletion of the ubiquitin pool within 30 minutes, no effect was observed for PAR-2 across the whole 60 minutes of the experiment, indicating lack of detectable ubiquitnation activity. A similar assay was performed using human UBC13/UEV1 as the E2, chosen based on PAR-2's sequence similarity to the E3 ligase RNF8, which works with this E2, however the result of this was also negative (data not shown).\\

\begin{figure}
\includegraphics[scale=1]{ubiquitin_assay}
\centering
\mycaption{PAR-2 does not show activity in an in vitro ubiquitination assay}{
\textbf{(A)} Size exclusion chromatography results for a sample of strep-tagged PAR-2 purified from insect cells. The two prominent bands in the 70-100 kDa region both contain PAR-2. Expected weight (with strep tag) = 75.24 kDa.
\textbf{(B)} Ubiquitination assay results for PAR-2 vs TRIM2. Whereas TRIM2 displays auto-polyubiquitination activity, as shown by the appearance of 'streak' at high masses representing polyubiquitinated protein, loss of unmodified TRIM2 and loss of free ubiquitin, no activity is observed for PAR-2 throughout the entire assay. 
}
\label{fig:ubiquitin_assay}
\end{figure}


\subsubsection{Targeted mutation to putative linchpin site has no phenotype}


A key site in many RING E3 ligases is an arginine or lysine residue immediately downstream of the final zinc-coordinating cysteine, which is known as the linchpin site. <what does this site do?>. 46\% of RING domains have an arginine at this site linchpin, whereas 14\% have lysine \citep{Stewart2017}. Typically, the choice of residue at this site regulates a trade-off between ubiquitination activity and E2 specificity. RINGs with a K at this site typically show lower ubiquitination activity in vitro but higher E2 specificity. \textcite{Stewart2017} showed for the protein BRCA1 that mutating this site from an arginine to a lysine increases ubiquitination activity but reduces E2 specificity. However, this isn't a universal mechanism, and many functional RING E3 ligases have other residues at this site <examples, refs>. This suggests that other mechanisms of <> must exist. Currently this is poorly understood.\\

In \textit{C. elegans} PAR-2 this site is a lysine, which suggests a potential role as a linchpin. Alignment of \textit{Caenorhabditis} PAR-2 RING domain sequences shows that this site is largely conserved as a lysine across the genus \cref{fig:linchpin_alignments}, but \textit{C. bovis}, \textit{C. castelli} and \textit{C. monodelphis} have neither an arginine nor a lysine at this site. The fact that this site isn't universally conserved may argue against an important role for ubiquitination activity, could suggest that PAR-2 does play an role as a ubiquitin ligase but doesn't rely on the linchpin, or could suggest that these other species have evolved alternative strategies.\\ 

\begin{figure}
\includegraphics[scale=0.9]{linchpin_alignments}
\centering
\mycaption{Linchpin site alignments}{
\textbf{(A)} Alignment for the PAR-2 linchpin site (red box) across species of the \textit{Caenorhabditis} genus. This site is largely conserved as a lysine, but some species do not follow this pattern.
\textbf{(B)} Alignment of the linchpin site across the TRIM family of ubiquitin ligase proteins. In most cases this is an arginine or a lysine, but in some cases a different amino acid is present.
}
\label{fig:linchpin_alignments}
\end{figure}

To test the potential role of linchpin-mediated autoubiquitination for PAR-2 membrane binding affinity, I used CRISPR to perform targeted mutation to this site, turning it into an alanine. <similar approach used in other studies>. As shown in \cref{fig:linchpin_in_vivo}, this has no detectable effect on membrane affinity. Whilst this cannot rule out a role for PAR-2 as a ubiquitin ligase in vivo, the result argues against a model in which linchpin-mediated autoubiquitination is a driver of high membrane association.\\

\begin{figure}
\includegraphics[scale=0.9]{linchpin_in_vivo}
\centering
\mycaption{Mutation of the PAR-2 linchpin site has no effect on membrane affinity}{
\textbf{(A)} Midplane images of wild type vs K94A GFP::PAR-2 at maintenance phase.
\textbf{(B)} Normalised cross-cortex profiles for wild type vs K94A PAR-2 averaged over the posterior domain.
\textbf{(C)} Membrane affinity measurements for wild type vs K94A PAR-2 in the posterior domain.
}
\label{fig:linchpin_in_vivo}
\end{figure}


\subsubsection{Discussion}

Whilst the results in this section do not provide evidence to support PAR-2 ubiquitin ligase activity, it's difficult to draw strong conclusions. Negative results in the ubiquitination assay could indicate lack of E3 ligase activity, but may have  also been caused by an inappropriate E2, damage to the protein during purification, or there could be a low level of activity that's below the detection threshold of the assay. The first point could be addressed by using an E2 scan, an approach that tests for activity using a wide array of available E2s, but this hasn't yet been explored. Negative results for the linchpin mutant could argue that PAR-2 doesn't act as an E3 ligase (at least not in a way that would impact membrane affinity), but given that the linchpin mechanism isn't universally conserved in E3s, ubiquitination activity cannot be ruled out.\\

% other possible ways of disrupting E2 interaction?

\clearpage
\subsection{Exploring a role for dimerisation}

Another key function attributed to RING domains is their ability to act as dimerisation domains. Many purified RING domains have been shown to self-associate in crystal structures, forming dimers or higher order oligomers. In most cases, dimerisation is achieved via hydrophobic interactions between short alpha-helical segments at the N and C termini of the core RING domain, which forms a four-helix bundle involving two RING monomers (fig x). This is stabilised by additional contacts in the core RING domain. <MORE HERE, pattern of hydrophobic residues etc.> \\

%RING domain dimerisation is typically mediated by a four-helix bundle, consisting of short alpha-helical segments N- and C-terminal to the core RING domain each of the two monomers. Dimeric RING domains have a characteristic pattern of hydrophobic residues within these regions, which are organised to form a hydrophobic surface in the 4-helix bundle, which stabilises dimerisation. 

In many cases, the significance of dimerisation is fundamentally linked to the  role of RING-domain proteins as ubiquitin ligases. RING dimerisation stabilises the closed E2-Ub conformation, by allowing each Ub to make simultaneous contacts with two RING domains, which in many cases this is essential for ubiquitination activity. Mutations to key hydrophobic residues in the helices flanking the core RING domain disrupt dimerisation and reduce ubiquitination activity (REFS). On the other hand, forced dimerisation of RINGs can hyperactivate ubiquitination activity, as has been shown for RNF4 (ref). \\

Whilst the majority of RING domains are dimeric in crystal structures, the oligomeric state in solution varies dramatically, ranging from entirely monomeric to constitutively dimeric. For example, whereas TRIM32 appears constitutively dimeric in solution \citep{Koliopoulos2016}, TRIM21 exists in a monomer-dimer equilibrium \citep{Dickson2018}, and TRIM25, whilst crystalising as a dimer, appears entirely monomeric in solution, only dimerising at very high concentrations \citep{Koliopoulos2016}. <comment on this, related to interface size?, deltaG?>.\\

In the case of RING domains at the weaker end of the dimerisation scale, which exist largely as monomers in isolation, the ability to dimerise may be context specific. RNF4 is largely monomeric at physiological concentrations in vitro, but is able to dimerise upon addition of its substrate. Dual binding of two RNF4 molecules to a single substrate molecule creates a locally high concentration of RNF4 that permits dimerisation \citep{Rojas-Fernandez2014}. <comment on this> Similarly, TRIM25 shows weak dimerisation ability on its own, but is stabilised by binding to an E2. \\

Notably, however, not all RING domains dimerise.  Some form higher order structures (e.g. the TRIM19 RING, which forms a ‘torus-shaped’ tetramer in crystal structures), whereas others lack any ability to oligomerise. The TRIM28 RING domain, for example, has a small N-terminal helix, but lacks a C-terminal helix, and forms no dimerisation contacts in crystal structures (ref). The TRIM56 RING domain lacks both helices, and similarly is monomeric in crystal structures (ref). Likewise, the CBL class of RING domains <more>..\\

Given the widespread roles of dimerisation <>, it’s not unlikely that RING domains have evolved functions as dimerisation domains in other contexts unrelated to ubiquitination….\\

In the case of PAR-2, the potential role for the RING domain as a dimerisation domain hasn't been tested, however full PAR-2 has been suggested to oligomerise. In vitro pull downs (\cite{Motegi2011}, \cite{Arata2016}) show that PAR-2 can weakly self-associate. \textcite{Arata2016} further showed by TIRF imaging that cortical PAR-2 exists as particles of varying size, which may represent a mix of monomers, dimers and possibly larger oligomers. In this section, I aim to test the dimerisation properties of PAR-2, and investigate the potential importance of this in vivo.\\

\subsubsection{PAR-2 RING domain sequence bears hallmarks of dimeric RINGs}

To investigate the potential role of the PAR-2 RING as a dimerisation domain, I began by analysing the sequence of the domain. I performed a BLAST search to compare the PAR-2 sequence against sequences of proteins for which the quaternary structure has been solved using laboratory methods, using the online SWISS model tool. This showed that the PAR-2 RING has sequence similarity to a number of well-characterised RING domains, most of which are dimeric in their crystal structures (table x). Sequence alignment shows that the PAR-2 sequence displays the general pattern of hydrophobic residues expected at the hydrophobic interface of dimeric RING domains (\cref{fig:alignments_other_rings}).\\

\begin{figure}
\includegraphics[scale=1]{alignments_other_rings}
\centering
\mycaption{Alignment of the PAR-2 RING domain N and C helices against similar proteins in the PDB}{
\textbf{(A)} N-helix alignment. Arrowhead indicates first zinc-coordinating cysteine of the core RING domain.
\textbf{(B)} C-helix alignment. Arrowhead indicates final zinc-coordinating cysteine of the core RING domain.
}
\label{fig:alignments_other_rings}
\end{figure}

These regions are also highly conserved in different \textit{Caenorhabditis} species (\cref{fig:alignments_par2}). Whilst there is some sequence variation, almost all species have an identical pattern of hydrophobic residues, which points to a potentially important role for PAR-2 RING domain dimerisation. \\

\begin{figure}
\includegraphics[scale=1]{alignments_par2}
\centering
\mycaption{Alignment of the PAR-2 RING domain N and C helices within the \textit{Caenorhabditis} genus}{
\textbf{(A)} N-helix alignment. Arrowhead indicates first zinc-coordinating cysteine of the core RING domain.
\textbf{(B)} C-helix alignment. Arrowhead indicates final zinc-coordinating cysteine of the core RING domain.
}
\label{fig:alignments_par2}
\end{figure}


\subsubsection{The PAR-2 RING domain displays concentration-dependent dimerisation in vitro}

\textit{Work in this section was performed with Dave Briggs in Neil McDonald's lab at the Crick}\\

The sequence analysis above suggests that the PAR-2 RING likely has the ability to dimerise. To directly test this, we aimed to purify a fragment of PAR-2 (40-120) containing the RING domain, including the putative dimerisation regions, and characterise its properties in vitro. We were successfully able to express the fragment in bacteria, and purify it by His pull down, followed by tag removal, ion exchange chromatography and size exclusion chromatography (see Methods).\\

To understand the oligomeric properties of the PAR-2 RING domain in solution, we subjected our purified sample to SEC-MALS, a common approach for characterising the molecular mass and oligomeric state of protein samples. The strategy follows a two step process. Firstly, size exclusion chromatography (SEC), separates particles in a sample based on their size. Larger particles leave the SEC column before smaller particles (i.e. at lower retention volumes). Refractive index scores monitor the concentration of particles in leaving the column. Secondly, the eluent flow leaving the SEC column is monitored continuously by a multi-angle light scattering (MALS) instrument, which provides an estimate of the molecular weight of particles based on light scattering properties.\\

We performed five SEC-MALS assays on PAR-2 RING, using sample concentrations of 10, 5, 2, 0.75 and 0.5 mg/ml (\ref{fig:sec_mals}A). In all cases a single peak is observed in refractive index traces, with corresponding molar mass measurements by MALS somewhere between the expected weight of a monomer and a dimer, suggesting that the sample exists in a monomer-dimer equilibrium. Furthermore, mass distributions show clear concentration-dependence, with more concentrated samples giving higher molar mass measurements. This is more clearly visualised in (\ref{fig:sec_mals}B), which shows that the average molecular weight of particles in the sample increases as a function of sample concentration. Overall, this data suggests that the PAR-2 RING domain displays concentration-dependent dimerisation, existing largely as a monomer at low concentrations and mostly dimeric at higher concentrations.\\

\begin{figure}
\includegraphics[scale=1]{sec_mals}
\centering
\mycaption{SEC-MALS titration reveals concentration-dependent dimerisation of the PAR-2 RING domain}{
\textbf{(A)} SEC-MALS traces for the PAR-2 RING domain at different concentrations. Solid lines indicate refractive index measurements, dotted lines indicate molar mass measurements. Colour coded by sample concentration. MALS traces are cropped to retention volumes for which refractive index is > 80\% of its peak value.
\textbf{(B)} Molecular weight measurements vs. concentration for the SEC-MALS assays in (A). Note, for samples in a monomer-dimer equilibrium, this represents the average weight of all particles in the sample. Error bars represent measurement uncertainty.
}
\label{fig:sec_mals}
\end{figure}


\subsubsection{Dimerisation is mediated by 4-helix bundle with hydrophobic core}

To better understand the structural basis of dimerisation, we aimed to resolve the structure of the PAR-2 RING domain dimer by X-ray crystallography. We successfully purified sufficient quantities of protein to set up crystal screens, but have so far been unsuccessful in generating crystals. Instead, I made use of recently developed tools for structure prediction. MORE. Alphafold. \citep{Jumper2021}.\\

I used AlphaFold to predict the structure of the PAR-2 RING domain dimer (residues 40-120 of the protein). The predicted structure (\cref{fig:ring_alphafold}) resembles the common structure of RING domain dimers, characterised by a four-helix bundle consisting of an N- and C- helix from each of the two monomers (\cref{fig:ring_alphafold}B). Within this bundle, we see a characteristic pattern of inwards facing hydrophobic residues (\cref{fig:ring_alphafold}D). Confidence scores, as measured by the AlphaFold metric pLDDT (predicted local distance difference test), are high throughout the structured region of the model (\cref{fig:ring_alphafold}C).\\


\begin{figure}
\includegraphics[scale=0.9]{ring_alphafold}
\centering
\mycaption{AlphaFold prediction for the 3D structure of the PAR-2 RING domain dimer}{
\textbf{(A)} 3D structure colour coded by chain.
\textbf{(B)} 3D structure with N and C helices highlighted.
\textbf{(C)} 3D structure colour coded by pLDDT, an AlphaFold metric for prediction confidence.
\textbf{(D)} 3D structure colour coded by hydrophobicity, with red indicating more hydrophobic and white indicating less hydrophobic. Inset shows enlarged view of the 4-helix bundle, showing internally facing hydrophobic residues.
}
\label{fig:ring_alphafold}
\end{figure}

Note, at the time of writing, whilst AlphaFold can model quaternary structures of oligomeric proteins, it relies on the user to input the expected oligomeric state (e.g. dimer in this case), and therefore cannot be used as a tool to predict oligomeric state. Therefore, the resulting structures should be interpreted as the best guess given that dimerisation is expected. Outputted models also do not provide the location of ligands (e.g. zinc ions in this case), but are expected to capture their effects on protein folding.\\

I also generated predictions for the PAR-2 RING across the \textit{Caenorhabditis} genus (\cref{fig:ring_alphafold_species}). These show that the 4-helix bundle structure is predicted to be largely conserved across the genus, although confidence scores within this region are variable. A major exception is \textit{C. latens}, which is predicted to form no bundle at all. This is compatible with the lack of hydrophobic residues at the key sites (\cref{fig:alignments_par2}). It's likely that, unlike other species, the RING domain of \textit{C. latens} PAR-2 is monomeric.\\

\begin{figure}
\includegraphics[scale=0.9]{ring_alphafold_species}
\centering
\mycaption{AlphaFold predictions for the PAR-2 RING domain across the \textit{Caenorhabditis} genus}{
Colour coded by pLDDT.
}
\label{fig:ring_alphafold_species}
\end{figure}

Given that dimerisation is predicted to be mediated by a hydrophobic interface within a 4-helix bundle, we would expect that mutations to hydrophobic residues within this region should disrupt dimerisation. This approach has previously been used <examples>. Unlike mutations to zinc-coordinating cysteines, which completely misfold the domain, mutations to this interface should preserve the tertiary structure of the domain and specifically disrupt dimerisation. I identified L109, predicted to be found in the C-helical region as a likely important site (\cref{fig:l109r_sec_mals}A), and L109R as a likely disruptive mutation. As we did for the wild-type RING domain, we expressed the mutant allele in bacteria and purified it using the same protocol. SEC-MALS shows that this mutant has the expected molecular weight of an entirely monomeric population (\cref{fig:l109r_sec_mals}A). By contrast, wild type RING at an equivalent concentration (\cref{fig:sec_mals}A, 0.75 mg/ml) is partially dimeric.\\

\begin{figure}
\includegraphics[scale=1]{l109r_sec_mals}
\centering
\mycaption{Mutation of residue at hydrophobic interface disrupts PAR-2 RING domain dimerisation}{
\textbf{(A)} Position of the L109 site within the 4-helix bundle of the PAR-2 RING domain. L109 site highlighted in blue, other residues coloured by hydrophobicity.
\textbf{(B)} SEC-MALS trace for PAR-2 RING (L109R). Solid line indicates refractive index, dotted line indicates molar mass. Sample concentration approx 0.75 mg/ml.
}
\label{fig:l109r_sec_mals}
\end{figure}

As a further test, we attempted to express and purify a C56S mutant RING domain for SEC-MALS. However, we found the protein to be unstable in vitro and were unable to purify it significant quantities to perform the assay, which is a common problem for unfolded proteins.\\

\subsubsection{Cytoplasmic PAR-2  does not self-associate in vivo}

The in vitro results described so far suggest that the PAR-2 RING domain displays concentration-dependent dimerisation. In vivo, PAR-2 concentrations are relatively low in the cytoplasm, but local concentrations are greatly enhanced upon membrane association. To investigate the potential role for RING-mediated PAR-2 dimerisation in vivo, I considered the cytoplasmic and membrane fractions separately, and devised two assays designed to test the dimeric state of PAR-2 in the cytoplasm and on the membrane.\\

To test the dimeric state of cytoplasmic PAR-2, I devised an in vivo pull-down assay based on dual labelling of PAR-2 with two different fluorophores (\cref{fig:tomm20_schematic}). Expressing PAR-2 with a mix of GFP and mCherry fluorophores means that, were cytoplasmic PAR-2 to form dimers, a mixed population of GFP/GFP, GFP/mCherry and mCherry/mCherry dimers would be expected to be present within the cytoplasm. Thus, forced relocalisation of GFP::PAR-2 to a particular location in the cell should result in a detectable signal mCherry::PAR-2 at that location. A similar assay was used previously by \textcite{Reich2019}, who demonstrated a stable interaction between PAR-6 and PKC-3, through forced localisation of GFP::PKC-3 to the plasma membrane with a plasma membrane tethered GFP binding protein (GBP). (In this case the plasma membrane was a suitable target because, under the conditions of the assay, PAR-6/PKC-3 do not normally bind to the plasma membrane). Here, I propose a similar approach, relying instead on recruitment to mitochondrial membranes within the cell. Mitochondrial membranes are dense within the \textit{C. elegans} zygote \citep{Dinkelmann2003}, and can be probed easily with a localisation signal from the TOMM-20 protein \citep{Watanabe2011}.\\

\begin{figure}
\includegraphics[scale=0.95]{tomm20_schematic}
\centering
\mycaption{Schematic of an in vivo dimerisation assay involving the mitochondrial probe TOMM-20}{
}
\label{fig:tomm20_schematic}
\end{figure}

To first test the potential utility of this method, I built a TOMM-20::GBP probe tagged with mKate, and introduced this into the worm by CRISPR. Crossing this line to a GFP::PAR-2 expressing line, and imaging the F1s (heterozygous for both GFP::PAR-2 and the probe), shows that the probe is well expressed, displays the expected mitochondrial localisation pattern, and is able to recruit GFP::PAR-2 to the mitochondria (\cref{fig:tomm20_merge}).\\

\begin{figure}
\includegraphics[scale=0.95]{tomm20_merge}
\centering
\mycaption{Mitochondrial GFP nanobody pulls GFP-tagged PAR-2 to mitochondria}{
Images of a zygote expressing Tomm20::GBP::mKate and GFP::PAR-2.
GFP channel image is not autofluorescence corrected. Shows a view between the midplane and cortex, as this gives a clearer signal for mitochondria.
}
\label{fig:tomm20_merge}
\end{figure}

To perform the proposed dimerisation assay, I next attempted to build an untagged version of the mitochondria probe (lacking mKate). Whilst I was successfully able to  introduce the construct into the genome of the worm, I found that this untagged probe failed to recruit any GFP::PAR-2 to the mitochondria, likely indicating that the probe failed to express. This suggests that the mKate tag may stabilising expression of the full construct, possibly due to the presence of introns within the mKate sequence. As an alternative method to free up the red channel, I introduced a point mutation into mKate by CRISPR designed to disrupt the chromophore region, which eliminates fluorescence \citep{Pletnev2008}. This construct, by contrast, was well expressed and, like the tagged parent construct, able to pull GFP::PAR-2 to the mitochondria. Next, I crossed this line to a line expressing mCherry::PAR-2 at the endogenous locus, homozygosing both constructs. I then crossed this line to a line homozygous for endogenous GFP::PAR-2, giving F1s heterozygous for GFP/mCherry PAR-2, and heterozygous for the probe.  Imaging embryos from these F1s showed that, whereas GFP::PAR-2 is fully pulled to the mitochondria, there is no detectable colocalisation with mCherry::PAR-2, which retains a normal pattern of cytoplasmic and cortical localisation (\cref{fig:tomm20_assay}). Overall, as mCherry signal is not observed at the mitochondria, the result implies that cytoplasmic PAR-2 is largely or entirely monomeric.\\

\begin{figure}
\includegraphics[scale=1]{tomm20_assay}
\centering
\mycaption{In vivo dimerisation assay shows that PAR-2 does not constitutively dimerise in vivo}{
Images of GFP::PAR-2 and mCherry::PAR-2 in a line heterozygous for GFP/mCherry PAR-2 and heterozygous for TOMM-20::GBP. GFP channel image is not autofluorescence corrected. Shows a view between midplane and cortex as this gives a clearer signal for mitochondria.
}
\label{fig:tomm20_assay}
\end{figure}


\subsubsection{Membrane-enriched PAR-2 self-associates in vivo}

I next aimed to test the ability of the PAR-2 RING to dimerise on the membrane, where concentrations are greatly enhanced compared to the cytoplasm. To investigate this, I used a pleckstrin homology (PH) plasma membrane probe to tether a RING-containing fragment of PAR-2 (1-177) to the plasma membrane. Should this construct be capable of dimerisation, we would expect some degree of dimerisation with endogenous PAR-2, and therefore preferential enrichment in the posterior of the cell. The construct is poorly expressed, but visible after autofluorescence correction by SAIBR. Whilst PH alone is uniform on the membrane, tethering it to the PAR-2 RING domain causes clear enrichment in the posterior, which suggests an interaction with endogenous PAR-2 (\cref{fig:ph_ring}). A C56S mutant construct, by contrast, shows a reduced degree of polarity compared to the wild type probe. Furthermore, the RING fragment alone shows no membrane localisation, suggesting that membrane enrichment is first required to permit dimerisation with endogenous PAR-2 (\cref{fig:ring_fragment_in_vivo}). This is in line with a previous study which showed a similar result \citep{Hao2006}.\\

\begin{figure}[!h]
\includegraphics[scale=1]{ph_ring}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Membrane-tethered RING domain fragment drives posterior localisation}{
\textbf{(A)} Schematic of the three constructs tested (GFP not shown).
\textbf{(B)} Midplane confocal images of the three constructs.
\textbf{(C)} Quantification of membrane to cytoplasmic ratios for the three constructs (Mean $\pm$ SD).
\textbf{(D)} Quantification of asymmetry index using the data in (C).
\textbf{(E)} Total expression of the three constructs. Expression of GFP::PH was downregulated by RNAi, aiming to roughly match expression of the other two constructs.
}
\label{fig:ph_ring}
\end{figure}

\begin{figure}
\includegraphics[scale=1]{ring_fragment_in_vivo}
\centering
\mycaption{PAR-2 RING domain fragment shows no membrane localisation}{
\textbf{(A)} Midplane image of mNG::PAR-2(1-177), showing entirely cytoplasmic localisation.
\textbf{(B)} Normalised cross-cortex profile of mNG::PAR-2(1-177) compared to mNG alone.
}
\label{fig:ring_fragment_in_vivo}
\end{figure}

These results suggest that the GFP::PH::RING construct is polarising by dimerising with polarised endogenous PAR-2. Therefore, it is expected that disrupting polarity of endogenous PAR-2 should disrupt this localisation. To test this, I disrupted endogenous polarity using RNAi against \textit{par-2} or \textit{par-6}, and visualised the effect on the GFP::PH::RING construct (\cref{fig:ph_ring_rnai}). In conditions of \textit{par-2} RNAi, endogenous PAR-2 is expected to be entirely absent from the cell, and aPARs mostly uniform. Under these conditions, the construct loses enrichment in the posterior, and overall polarity is low. Similarly, in conditions of \textit{par-6} RNAi, where endogenous PAR-2 is expected to be uniform on the membrane, albeit less weakly concentrated, the construct also loses the ability to enrich in the posterior. Whilst we might expect overall affinity to be higher in the \textit{par-6} RNAi condition, as there is some level of endogenous PAR-2 on the membrane, this doesn't appear to be the case in this assay. This may imply a concentration dependent effect, whereby the construct is only able to respond to endogenous PAR-2 that's highly concentrated on the membrane.\\

\begin{figure}
\includegraphics[scale=1]{ph_ring_rnai}
\centering
\mycaption{Localisation of membrane-tethered RING domain is dependent on endogenous PAR patterns}{
\textbf{(A)} Midplane confocal images of GFP::PH::RING under conditions of no RNAi, \textit{par-2} RNAi and \textit{par-6} RNAi.
\textbf{(B)} Quantification of membrane to cytoplasmic ratios for the three constructs (Mean $\pm$ SD).
\textbf{(C)} Quantification of asymmetry index using the data in (C).
}
\label{fig:ph_ring_rnai}
\end{figure}


\subsubsection{Mutation to dimerisation interface weakens membrane association in vivo}

Overall, the data described so far suggests a mechanism whereby cytoplasmic PAR-2 is monomeric, and enrichment on the membrane switches PAR-2 to a dimeric state via a concentration-dependent dimerisation reaction. I next aimed to investigate the potential importance of this dimerisation reaction in vivo. As discussed previously, unfolding the RING domain by targeted mutation to a zinc-interacting cysteine has a strong phenotype in vivo, characterised by a strong reduction in membrane affinity. However, as C56S mutation is expected to disrupt the tertiary structure of the whole domain, rather than just the dimerisation interface, it may also have additional effects. Structure prediction analysis and SEC-MALS data indicate that a single mutation to a hydrophobic residue in the putative dimerisation interface (L109R) is sufficient to specifically disrupt dimerisation. To investigate the significance of PAR-2 dimerisation in vivo, I performed targeted mutations to this site by CRISPR. In addition to L109R, I performed an additional L to R mutation to L50, which is found in the N-helix, and doesn't interact with L109 (\cref{fig:l50_structure}).\\

\begin{SCfigure}
\includegraphics[scale=0.8]{l50_structure}
\mycaption{Position of the L50 site within the 4-helix bundle of the PAR-2 RING domain}{L50 site highlighted in blue, other residues coloured by hydrophobicity}
\label{fig:l50_structure}
\end{SCfigure}

As shown in \cref{fig:dimer_interface_mutants_in_vivo}, both mutations cause a strong reduction in membrane affinity. The effect is stronger for L109R than L50R, which may suggest that L50R has a weaker disruptive effect on dimerisation. Combination of both mutants has no effect compared to L109R alone, which implies that dimerisation is completely disrupted in L109R. Interestingly, however, neither mutant reduces membrane affinity to the same extent as C56S. Assuming that dimerisation is fully disrupted by L109R, this implies that fully unfolding the domain may have additional effects compared to just disrupting dimerisation alone. One plausible explanation is that, in the case of C56S, the unfolded domain interferes with normal membrane binding, and therefore has an additional negative effect on membrane affinity. Compatible with this, a PH::RING fusion has lower overall membrane affinity than PH alone (\cref{fig:ph_ring}). Combining C56S and L109R has no effect compared to C56S alone, suggesting that dimerisation is completely disrupted in the C56S mutant.\\

\begin{figure}
\includegraphics[scale=0.9]{dimer_interface_mutants_in_vivo}
\centering
\mycaption{Dimerisation interface mutants disrupt PAR-2 membrane association}{
\textbf{(A)} Midplane confocal images of wild type, L50R, L109R and C56S PAR-2.
\textbf{(B)} Normalised cross-cortex profiles for the four PAR-2 alleles in (A).
\textbf{(C)} Quantification of membrane to cytoplasmic ratio for the four mutants and (A) and (B) and two additional combined mutants.
}
\label{fig:dimer_interface_mutants_in_vivo}
\end{figure}


\subsubsection{Discussion}

% comment on arata paper

% Plausibly, if autoubiquitination does play a role, and this is dimerisation dependent, then dimerisation interface mutants could would also work through this mechanism.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\section{A thermodynamic model of PAR-2 dimerisation}

\textit{This section describes work performed in conjunction with David Zwicker}

\subsection{Introduction}

\subsection{Model description}

%Might be worth having a glossary of terms: ideal solution, chemical potential, enthalpy, chemical system, chemical state, thermodynamic equilibrium, free energy, volume fraction

% Thermodynamic equilibrium puts constraints on the behaviour of a system, and allows us to accurately describe behaviour with a small number of easily interpretable parameters. In a multi-component chemical system, thermodynamic equilibrium is reached when all chemical components are at concentrations that have no tendancy to change with time, so that there are no overall changes in the properties of the system.

% Systems are described by a number of chemical components, each of which has an associated chemical potential
% Chemical potential describes
% Thermodynamic equilibrium is reached when all chemical components are at concentrations that have no tendancy to change with time, so that there are no overall changes in the properties of the system. This state is reached when the chemical potentials describing each species are all identical, such that movement of a particle between two states leads to no net change in overall energy of the system.


Consider, for example a simple chemical system in which material is exchanging between two states, $a$ and $b$:
\begin{center}
\ce{a <=> b}
\end{center}

Regardless of how the system is initiated, the system will eventually stabilise in a state where the forward and reverse reactions are equal, meaning that the concentrations of a and b will have no tenancy to change over time. Energetically speaking, this state is reached when the total free energy of the system ($F$) is minimised. When our system is at this minimum, the partial derivative of total energy with respect the the concentration of a must be equal to the equivalent partial derivative of b, meaning that any a to b reaction (or vice versa) will have no net impact on the overall energy energy of the system:
\begin{align}
\frac{dF}{da} = \frac{dF}{db}
\end{align}

These derivatives are more commonly referred to as the chemical potential of each species ($\mu_x$). Using ideal solution theory, this property can be described for each species, in $k_BT$ units, as:
\begin{align}
\mu_a &= \ln\phi_a - w_a\\
\mu_b &= \ln\phi_b - w_b
\end{align} 

where $\phi_a$ and $\phi_b$ represent the volume fraction occupied by species a and b respectively, and $w_a$ and $w_b$ are enthalpies associated with each state (a measure of how energetically favourable each state is). Since the total amount of material is conserved, we have the initial constraint that:
\begin{equation}
\phi_a + \phi_b = \phi
\end{equation}

where $\phi$ is a constant representing the total amount of material. As stated earlier, chemical equilibrium is reached when $\mu_a = \mu_b$. Thus, for a given total amount of protein ($\phi$), we can use our thermodynamic description to calculate $\phi_a$ and $\phi_b$ at equilibrium:
\begin{align}
\phi_a &= \frac{\phi}{1 + e^{w_b - w_a}}\\
\phi_b &= \frac{\phi}{1 + e^{w_a - w_b}}
\end{align}

We can also see from equations xxx that, at equilibrium, the following relationship must always we true:
\begin{equation}
\frac{\phi_a}{\phi_b} = e^{w_a - w_b}
\end{equation}

showing that, regardless of the total amount of protein, there is a constant ratio between the two states at equilibrium. (This is, of course, obvious just by looking at the chemical equation). The ratio that is reached will depend on the enthalpy terms associated with each state. Where $w_a > w_b$ more protein will end up in state a than state b, and vice versa (\cref{fig:thermodynamic_simple_example2}).\\

\begin{figure}[!h]
\includegraphics[scale=0.9]{thermodynamic_simple_example2}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:thermodynamic_simple_example2}
\end{figure}

A similar description can apply to a species that is exchanging between two physical compartments. In this case a and b can represent protein in one of two spatial compartments, with $w_a$ and $w_b$ representing enthalpies associated with each compartment. The only difference is that the conservation of mass term must be slightly changed to account for potential differences in the volume of the two compartments. We will use:
\begin{equation}
\phi_a + \alpha\phi_b = \phi
\end{equation}

where $\alpha$ is a dimensionless exchange factor comparing the volume of the two compartments. (In this case $\phi$ can be interpreted as the volume fraction when all protein is in compartment a). As before, we end up with a linear relationship between concentrations in the two compartments, with ratios specified by $w_a - w_b$ (\cref{fig:thermodynamic_simple_example}) - in this case, for simplicity, $\alpha = 1$).

\begin{figure}[!h]
\includegraphics[scale=0.9]{thermodynamic_simple_example}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:thermodynamic_simple_example}
\end{figure}

\subsubsection{Thermodynamic description of membrane binding}

We can use the same description for a particle exchanging between membrane bound and cytoplasmic pools (m and c), by considering membrane and cytoplasmic compartments as two volume compartments. In this case the membrane compartment can be described as a thin volume, characterised by a surface area and a (small) thickness. Since, as shown above, equilibrium concentrations are specified by the difference in enthalpy terms (rather than their absolute values), we can set $w_c = 0$, and define the chemical potentials as:
\begin{align}
\mu_c &= \ln\phi_c\\
\mu_m &= \ln\phi_m - w_m
\end{align} 

where $w_m$ can be interpreted as the membrane insertion energy (i.e. membrane affinity). The total amount of material is described as:
\begin{equation}
\phi_c + \alpha\phi_m = \phi
\end{equation}

where $\alpha$ (the volume ratio between the compartments) will be some small number $<<1$. Thus, as before, membrane concentrations at equilibrium can be described as a linear function of cytoplasmic concentrations, the slope of which will depend on $w_m$.\\

\subsubsection{Thermodynamic description of dimerisation}

We can extend the same analysis to consider a species dimerising:

\begin{center}
\ce{2x_1 <=> x_2}
\end{center}

Chemical potentials are can be described as:
\begin{align}
\mu_{x,1} &= \ln\phi_{x,1}\\
\mu_{x,2} &= \ln\phi_{x,2} - w_d
\end{align} 

with $w_d$ represents the dimerisation energy. Thermodynamic equilibrium is reached when $2\mu_{x,1} = \mu_{x,2}$ (the factor of 2 accounting for the differing stoichiometry between the two species), leading to the following relationship:
\begin{equation}
\frac{\phi_{x,2}}{(\phi_{x,1})^2} = e^{w_d}
\end{equation}

Thus, concentrations of dimeric protein will vary linearly with the square of monomer concentrations (\cref{fig:thermodynamic_simple_dimer}A) (again, this is obvious from the chemical equation), and so the fraction of protein in the dimeric state will increase as total protein amounts increase (\cref{fig:thermodynamic_simple_dimer}B). Additionally, with the constraint that $\phi_x = \phi_{x,1} + \phi_{x,2}$, we can derive relationships for the concentrations of monomer and dimer as a function of total protein (check):
\begin{align}
\phi_{x,1} &= \frac{1}{2}e^{-w_d}\left(\sqrt{4e^{w_d}\phi_x + 1} - 1\right)\\
\phi_{x,2} &= \phi_x - \frac{1}{2}e^{-w_d}\left(\sqrt{4e^{w_d}\phi_x + 1} - 1\right)
\end{align}

\begin{figure}[!h]
\includegraphics[scale=1]{thermodynamic_simple_dimer}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:thermodynamic_simple_dimer}
\end{figure}


\subsubsection{4-species dimerisation model}

I next consider the case in which both dimerisation and membrane exchange are occurring. In this case, there are four potential species to consider: membrane monomers, membrane dimers, cytoplasmic monomers and cytoplasmic dimers (\cref{fig:thermodynamic_model_species}A). We can describe chemical potentials for the four species as:
\begin{align}
\mu_{c,1} &= \ln(\phi_{c,1}) - w_{c,1}\\
\mu_{c,2} &= \ln(\phi_{c,2}) - w_{c,2}\\
\mu_{m,1} &= \ln(\phi_{m,1}) - w_{m,1}\\
\mu_{m,2} &= \ln(\phi_{m,2}) - w_{m,2}
\end{align}

where $w_x$ represents the energy for each species. As only the differences in energies are important we set $w_{c,1} = 0$, and describe the four energies as:
\begin{align}
w_{c,1} &= 0\\
w_{c,2} &= w_d\\
w_{m,1} &= w_m\\
w_{m,2} &= 2w_m + w_d
\end{align}

where $w_d$ represents the energy gain upon dimerisation and $w_m$ the energy gain upon membrane association. Whilst experimental data argues against significant dimerisation in the cytosol, there's no reason the believe that PAR-2 has an inability to dimerise in cytosolic conditions (in fact, we know from SEC-MALS that pure RING can dimerise at without a membrane). Instead, it seems likely that concentrations in the cytoplasm are just too low for this to happen. Therefore, in the interest of simplicity, we here assume that the dimerisation energy is equal in the membrane and cytoplasm. We also assume that the membrane association energy of a dimer is twice that of a monomer (as dimers have two binding interfaces, one for each monomer). Thermodynamic equilibrium is reached when:
\begin{equation}
2\mu_{c,1} = \mu_{c,2} = 2\mu_{m,1} = \mu_{m,2}
\end{equation}

Since the total amount of material is conserved, we have:
\begin{align}
\phi &= \phi_c + \alpha\phi_m
\end{align}

where $\phi_c = \phi_{c,1} + \phi_{c,2}$ and $\phi_m = \phi_{m,1} + \phi_{m,2}$. With constraints imposed by mass conservation, equation x can be solved analytically to obtain expressions for each of the four species as a function of $\phi$. Figure \ref{fig:thermodynamic_model_species} shows that, for a fixed total amount of protein, the equilibrium fraction of total protein in each of the four states will vary depending on the two energy parameters. With low $w_d$ and low $w_m$ protein will largely by cytoplasmic and monomeric; with low $w_d$ and high $w_m$ protein will be largely membrane-bound and monomeric; with high $w_d$ and low $w_m$ protein will be largely cytoplasmic and dimeric; and with high $w_d$ and high $w_m$ protein will be largely membrane-bound and dimeric. At intermediate energies a mix of states can be observed.\\

\begin{figure}[!h]
\includegraphics[scale=0.9]{thermodynamic_model_species}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:thermodynamic_model_species}
\end{figure}


\subsection{Intermediate dimerisation can drive nonlinear membrane association}

Given the nonlinearities associated with dimerisation reactions, the distribution of protein between the four states is expected to vary depending on total protein amounts. This is particularly interesting in the case of PAR-2, as partitioning between the cytoplasm and membrane doesn’t follow simple linear kinetics, but varies depends on dosage (section x). I therefore investigated whether our simple thermodynamic behaviour can capture similar behaviour, by varying $\phi$ across a range of concentrations and solving for $\phi_m$ and $\phi_c$. \\

Figure \ref{fig:thermodynamic_model_feedback} shows that, in certain parameter regimes, clear nonlinearity can be observed in the relationship between cytoplasmic and membrane concentrations. For a given value of $w_m$, changing $w_d$ impacts both the shape and the amplitude of the relationship (\cref{fig:thermodynamic_model_feedback}A). By stabilising the high affinity dimeric state, increases to $w_d $increase overall membrane affinity for a constant $w_m$, leading to a steeper relationship. \\

\begin{figure}[!h]
\includegraphics[scale=1]{thermodynamic_model_feedback}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:thermodynamic_model_feedback}
\end{figure}

As shown in \cref{fig:thermodynamic_model_feedback}B, the effective exponent of the cytoplasm vs membrane relationship ($\beta$), obtained by fitting $\phi_m = \alpha \phi_c^{\;\beta}$, varies across parameter space between 1 (linear relationship) and 2 (quadratic relationship). Peak nonlinearity occurs in regions of high membrane energy and intermediate dimerisation energy. This corresponds to regions of parameter space in which membrane protein is largely dimeric but cytoplasmic protein largely monomeric, across the relevant range of dosages (\cref{fig:thermodynamic_model_dimer_fractions}). Where $w_d$ is low, protein is unable to dimerise, even at enriched membrane concentrations, and so the model follows simple linear kinetics like the description in section x. If, on the other hand, $w_d$ is too high, protein is constitutively dimeric in both the membrane and cytoplasm, and will behave similarly to a monomer without nonlinearities (albeit with a higher membrane affinity). If $w_d$ is intermediate, then an asymmetry is observed whereby protein is dimeric on the membrane but not in the cytoplasm, provided that $w_m$ is sufficiently large to set up a substantial concentration difference between the two compartments. NEED A COMMENT ON WHY THIS GIVES NONLINEARITY\\

\begin{figure}[!h]
\includegraphics[scale=0.9]{thermodynamic_model_dimer_fractions}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:thermodynamic_model_dimer_fractions}
\end{figure}


\clearpage
\subsection{Assessment of PAR-2 membrane binding kinetics}

Overall, a few behaviours of the model make it highly relevant for consideration of PAR-2. Firstly, the model suggests that a reduction in dimerisation energy ($w_d$), as would be expected for a RING-disrupting mutant, can substantially decrease membrane association, independently of the membrane association energy ($w_m$). Secondly, the model suggests that, in some parameter regimes, a nonlinear relationship can occur between cytoplasmic and membrane concentrations. This latter point has the condition that dimerisation be supported in the membrane but not the cytoplasm, both of which appear to be the case for PAR-2.\\

To assess how well the membrane-binding kinetics of wild type and RING-mutant PAR-2 can be captured by this model, I performed a quantitative comparison to in vivo data of cytoplasmic vs membrane concentrations. In light of evidence described in the previous chapter, which suggests that C56S may disrupt membrane binding by more than just disrupting dimerisation, I decided to repeat the experiment shown in fig x using the L109R mutant as a baseline instead of C56S. The results show a similar effect, whereby L109R mutant PAR-2 follows a different trajectory to wild type.\\

I then fit this data to a dimerisation model in which $w_m$ is unchanged between the two conditions but $w_d$ allowed to change, implying that the dimerisation interface mutant is expected to disrupt the dimerisation interaction without changing the membrane interaction. Figure x shows that the data can be closely captured by a model in which L109R mutation causes a decrease in $w_d$ with a constant $w_m$. Note that the dimerisation constant (kd) is exponentially related to the dimerisation energy, so a x decrease in wd corresponds to a x fold difference in kd. Figure x shows a histogram of parameter values obtained by fitting bootstrapped samples of the data to the model. This shows that, whilst there is some uncertainty in the parameter values, particularly $w_d$, there is clear separation between the two conditions.\\

\begin{figure}[!h]
\includegraphics[scale=1]{thermodynamic_model_wt_vs_l109r}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:thermodynamic_model_wt_vs_l109r}
\end{figure}


\subsection{Discussion}

% We have also considered non-ideal models and this doesn't change the behaviour within concentration ranges relevant to the in vivo measurements

The question remains as to why C56S appears more disruptive than L109R. One possibility is that dimerisation isn’t completely disrupted in L109R, although if this were the case, I’d expect L109R and L50R to be additive, which they are not. Another possibility is that C56S mutation, by causing the entire domain to unfold, has an additional negative impact on membrane binding whereby the unfolded domain interferes with the association between the polybasic domain and the membrane (effectively causing a decrease in $w_m$ in addition to $w_d$).\\

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\section{Modelling dimerisation-driven positive feedback in the PAR network}

\subsection{Model description}

\subsubsection{Reducing thermodynamic model to a two-species model} 

To simplify further analysis, we can convert the four species thermodynamic model to a two species model, in which we only track overall membrane and cytoplasmic concentrations, $\phi_m$ and $\phi_c$, irrespective of dimeric state. If we assume that dimerisation reactions in the membrane and cytoplasm are fast relative to membrane binding, we can define instantaneous monomer and dimer concentrations as a function of overall concentration. For example, for cytoplasmic protein:
% check equations
\begin{align}
\phi_{c,1} &= \frac{1}{2}e^{-w_d}\left(\sqrt{4e^{w_d}\phi_c + 1} - 1\right)\\
\phi_{c,2} &= \phi_c - \frac{1}{2}e^{-w_d}\left(\sqrt{4e^{w_d}\phi_c + 1} - 1\right)
\end{align}

which is identical to the description in section x. As a result, and because we can describe the chemical potential of each of these species individually (equations xxx), we can combine expression <> and <> to define the overall chemical potential of cytoplasmic protein ($\mu_c$) as a function of overall cytoplasmic concentrations ($\phi_c$):
\begin{equation}
\mu_c = \ln(\phi_c) - \frac{1}{2}\ln\left(1 + 2e^{w_d}\phi_c + \sqrt{4e^{w_d}\phi_c + 1}\right)
\end{equation}
% check equation

And analogously for membrane bound protein: 
\begin{equation}
\mu_m = \ln(\phi_m) - \frac{1}{2}\ln\left(1 + 2e^{w_d}\phi_m + \sqrt{4e^{w_d}\phi_m + 1}\right) - w_m
\end{equation}
% check equation

These two equations can be solved analytically at equilibrium ($\mu_c$ = $\mu_m$) to find equilibrium membrane and cytoplasmic concentrations.


\subsubsection{Explicit description of membrane binding kinetics} 

Building kinetic models requires a description of membrane and unbinding rates, rather than just energetics. Exchange can be written as:

\begin{center}
\ce{$\phi_c$ <=>[$k_{on}$][$k_{off}$] $\phi_m$}
\end{center}

For a given composition, flux on ($s_{on}$) is equal to $k_{on}\phi_c$, and flux off ($s_{off}$) is equal to $k_{off}\phi_m$. Using transition state theory, these fluxes can be written as:
\begin{align}
s_{on} &= ke^{\mu_c}\\
s_{off} &= ke^{\mu_m}
\end{align}

where $k$ is a kinetic factor. Using our previous descriptions of cytoplasmic and membrane chemical potentials, the kinetic rate constants can be calculated as:
\begin{align}
k_{on} &= \frac{k}{\sqrt{2e^{w_d}\phi_c+ 1}}\\
k_{off} &= \frac{ke^{-w_m}}{\sqrt{2e^{w_d}\phi_m+ 1}}
\end{align}

implying a dependence on both energies and concentrations. This is shown for $k_{off}$ in fig x. Fig xA shows off rates plot a function of membrane energy and dimerisation energy, for a given membrane concentration. We can also see (fig xB), the dependence of off rates on concentrations.

\begin{figure}[!h]
\includegraphics[scale=1]{thermodynamic_model_koff}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:thermodynamic_model_koff}
\end{figure}

\subsubsection{Incorporation into mutual antagonism model}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\section{Experimental manipulation of PAR-2 dimerisation}

\subsection{Introduction}

\subsection{Forced constitutive dimerisation of PAR-2 shifts localisation towards lower-affinity membranes}

% Strikingly, we see the unexpected appearance of visible structures within the cell. Based on comparison to RAB localisation patterns, these appear to be endosomes.

% We also see the appearance of PAR-2 on the anterior membrane, suggesting that the protein is less susceptable to antagonism by PKC-3

\begin{figure}[!h]
\includegraphics[scale=1]{gcn4}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:gcn4}
\end{figure}

\begin{figure}[!h]
\includegraphics[scale=1]{gcn4_quantification}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:gcn4_quantification}
\end{figure}

\begin{figure}[!h]
\includegraphics[scale=1]{gcn4_alone}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:gcn4_alone}
\end{figure}

\clearpage
\subsection{Incorporating internal membranes into the thermodynamic model}

I consider two further species, representing monomers and dimers bound to internal membranes.  Concentrations are specified by $\phi_{i,1}$ and $\phi_{i,2}$, chemical potentials $\mu_{i,1}$ and $\mu_{i,2}$ with energy parameters $w_{i,1}$ and $w_{i,2}$, where:

\begin{align}
w_{i,1} &= w_i\\
w_{i,2} &= 2w_i + w_d
\end{align}

where $w_i$ is the internal membrane insertion energy. Thermodynamic equilibrium is reached when:
\begin{equation}
2\mu_{c,1} =  \mu_{c,2} = 2\mu_{p,1} =  \mu_{p,2} = 2\mu_{i,1} =  \mu_{i,2}
\end{equation}

The total amount of protein is conserved according the the following:
\begin{equation}
\overline{\phi} = \phi_c + \alpha\phi_p + \beta\phi_i
\end{equation}

where $\alpha$ is a non-dimensional conversion factor between cytoplasmic and plasma membrane volume fractions, $\beta$ is the equivalent factor for internal membranes. Setting $\overline{\phi}$ as a fixed value, $\phi_{c,1}$, $\phi_{c,2}$, $\phi_{m,1}$, $\phi_{m,2}$, $\phi_{i,1}$ and $\phi_{i,2}$ can be solved numerically. Using this model, we can investigate the relationship between dimerisation energy ($w_d$) and PAR-2 localisation in systems with varying levels of internal membrane charge (varying $w_i$). As shown in fig x, we can see that, in systems where $w_i$ is sufficiently high, constitutive dimerisation (high $w_d$) can result in considerable PAR-2 levels on internal membranes, which isn't observed at lower/intermediate dimerisation strengths. 

\begin{figure}[!h]
\includegraphics[scale=0.90]{six_species_thermodynamic}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:six_species_thermodynamic}
\end{figure}

% Whilst increasing wd may increase binding to internal membranes (provided that wi is high enough), this should always be accompanied by an increase in levels at the plasma membrane. In other words, we cannot get a scenario akin to what we see in vivo,  whereby constitutive dimerisation actually reduces levels on the plasma membrane.


\subsection{Modelling kinetics outside of equilibrium}


\begin{figure}[!h]
\includegraphics[scale=0.95]{three_surface_kinetic}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:three_surface_kinetic}
\end{figure}


\subsection{aPAR disruption partially restores expected equilibrium localisation pattern}

\begin{figure}[!h]
\includegraphics[scale=0.95]{gcn4_par3mut}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:gcn4_par3mut}
\end{figure}

\subsection{Discussion}



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\section{Discussion}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\section{Materials and methods}


\subsection{Worm maintenance}
\subsection{Generating transgenic lines by CRISPR}

\begin{figure}[!h]
\includegraphics[scale=1]{glh}
\setlength{\abovecaptionskip}{20pt}
\centering
\mycaption{Title}{Caption}
\label{fig:glh}
\end{figure}



\subsection{Microscopy}
\subsection{Full-length PAR-2 purification}
\subsection{Ubiquitination assays}
\subsection{PAR-2 RING domain fragment purification}
\subsection{SEC-MALS}
\subsection{Image analysis}
\subsection{Modelling methods}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\section{Appendix: Analysis of PAR-2 diffusion kinetics by single particle tracking}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\section{Abbreviations}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\clearpage
\section{Bibliography}

\printbibliography

\end{document}
